[2019-10-01 13:22:34,477] : [INFO] : [bleServerLogger] : [bleServer.py] : [getBluetoothSocket] : Bluetooth server socket successfully created for RFCOMM service...
[2019-10-01 13:22:34,478] : [INFO] : [bleServerLogger] : [bleServer.py] : [getBluetoothConnection] : Bluetooth server socket bind successfully on host  to PORT_ANY...
[2019-10-01 13:22:34,478] : [INFO] : [bleServerLogger] : [bleServer.py] : [getBluetoothConnection] : Bluetooth server socket put to listening mode successfully ...
[2019-10-01 13:22:34,478] : [INFO] : [bleServerLogger] : [bleServer.py] : [getBluetoothConnection] : Waiting for connection on RFCOMM channel 1
[2019-10-01 13:22:34,478] : [ERROR] : [bleServerLogger] : [bleServer.py] : [advertiseBluetoothService] : Failed to advertise bluetooth services  ... 
Traceback (most recent call last):
  File "bleServer.py", line 77, in advertiseBluetoothService
    profiles = [ SERIAL_PORT_PROFILE ],
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 176, in advertise_service
    raise BluetoothError (str (e))
BluetoothError: (2, 'No such file or directory')
[2019-10-01 13:26:04,319] : [ERROR] : [bleServerLogger] : [bleServer.py] : [acceptBluetoothConnection] : Failed to accept bluetooth connection ... 
Traceback (most recent call last):
  File "bleServer.py", line 86, in acceptBluetoothConnection
    self.clientSocket, clientInfo = self.serverSocket.accept()
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 129, in accept
    client, addr = self._sock.accept ()
KeyboardInterrupt
[2019-10-01 13:26:04,322] : [ERROR] : [bleServerLogger] : [bleServer.py] : [deserializedData] : Failed to de-serialized string ... 
Traceback (most recent call last):
  File "bleServer.py", line 113, in deserializedData
    self.dataObj=pickle.loads(_dataRecv)
TypeError: loads() argument 1 must be string, not None
[2019-10-01 13:26:04,323] : [INFO] : [bleServerLogger] : [bleServer.py] : [writeJsonFile] : text.json file successfully opened to <open file 'text.json', mode 'w' at 0x7fe4f4ba7810>
[2019-10-01 13:26:04,324] : [INFO] : [bleServerLogger] : [bleServer.py] : [writeJsonFile] : Content dumped successfully to the text.json file
[2019-10-01 13:26:04,325] : [INFO] : [bleServerLogger] : [bleServer.py] : [writeJsonFile] : text.json file successfully closed
[2019-10-01 13:26:04,326] : [ERROR] : [bleServerLogger] : [bleServer.py] : [closeBluetoothSocket] : Failed to close the bluetooth sockets 
Traceback (most recent call last):
  File "bleServer.py", line 135, in closeBluetoothSocket
    self.clientSocket.close()
AttributeError: 'NoneType' object has no attribute 'close'
[2019-10-01 13:33:58,251] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-01 13:33:58,251] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-01 13:33:58,251] : [ERROR] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Couldn't find the RaspberryPi Bluetooth service : Invalid uuid
Traceback (most recent call last):
  File "bleClient.py", line 51, in getBluetoothServices
    bleService = find_service( uuid = self.uuid, address = self.addr )
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 186, in find_service
    devices = discover_devices ()
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 22, in discover_devices
    raise BluetoothError ("error communicating with local "
BluetoothError: error communicating with local bluetooth adapter
[2019-10-01 13:33:58,252] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-01 13:51:06,418] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-01 13:51:06,418] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-01 13:51:06,418] : [ERROR] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Couldn't find the RaspberryPi Bluetooth service : Invalid uuid
Traceback (most recent call last):
  File "bleClient.py", line 51, in getBluetoothServices
    bleService = find_service( uuid = self.uuid, address = self.addr )
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 186, in find_service
    devices = discover_devices ()
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 22, in discover_devices
    raise BluetoothError ("error communicating with local "
BluetoothError: error communicating with local bluetooth adapter
[2019-10-01 13:51:06,419] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-01 13:51:41,712] : [INFO] : [bleServerLogger] : [bleServer.py] : [getBluetoothSocket] : Bluetooth server socket successfully created for RFCOMM service...
[2019-10-01 13:51:41,713] : [INFO] : [bleServerLogger] : [bleServer.py] : [getBluetoothConnection] : Bluetooth server socket bind successfully on host  to PORT_ANY...
[2019-10-01 13:51:41,713] : [INFO] : [bleServerLogger] : [bleServer.py] : [getBluetoothConnection] : Bluetooth server socket put to listening mode successfully ...
[2019-10-01 13:51:41,713] : [INFO] : [bleServerLogger] : [bleServer.py] : [getBluetoothConnection] : Waiting for connection on RFCOMM channel 1
[2019-10-01 13:51:41,713] : [ERROR] : [bleServerLogger] : [bleServer.py] : [advertiseBluetoothService] : Failed to advertise bluetooth services  ... 
Traceback (most recent call last):
  File "bleServer.py", line 77, in advertiseBluetoothService
    profiles = [ SERIAL_PORT_PROFILE ],
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 176, in advertise_service
    raise BluetoothError (str (e))
BluetoothError: (2, 'No such file or directory')
[2019-10-01 14:03:59,685] : [INFO] : [bleServerLogger] : [bleServer.py] : [getBluetoothSocket] : Bluetooth server socket successfully created for RFCOMM service...
[2019-10-01 14:03:59,686] : [INFO] : [bleServerLogger] : [bleServer.py] : [getBluetoothConnection] : Bluetooth server socket bind successfully on host  to PORT_ANY...
[2019-10-01 14:03:59,686] : [INFO] : [bleServerLogger] : [bleServer.py] : [getBluetoothConnection] : Bluetooth server socket put to listening mode successfully ...
[2019-10-01 14:03:59,687] : [INFO] : [bleServerLogger] : [bleServer.py] : [getBluetoothConnection] : Waiting for connection on RFCOMM channel 1
[2019-10-01 14:03:59,687] : [ERROR] : [bleServerLogger] : [bleServer.py] : [advertiseBluetoothService] : Failed to advertise bluetooth services  ... 
Traceback (most recent call last):
  File "bleServer.py", line 77, in advertiseBluetoothService
    profiles = [ SERIAL_PORT_PROFILE ],
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 176, in advertise_service
    raise BluetoothError (str (e))
BluetoothError: (2, 'No such file or directory')
[2019-10-01 14:08:39,969] : [ERROR] : [bleServerLogger] : [bleServer.py] : [acceptBluetoothConnection] : Failed to accept bluetooth connection ... 
Traceback (most recent call last):
  File "bleServer.py", line 86, in acceptBluetoothConnection
    self.clientSocket, clientInfo = self.serverSocket.accept()
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 129, in accept
    client, addr = self._sock.accept ()
KeyboardInterrupt
[2019-10-01 14:08:39,971] : [ERROR] : [bleServerLogger] : [bleServer.py] : [deserializedData] : Failed to de-serialized string ... 
Traceback (most recent call last):
  File "bleServer.py", line 113, in deserializedData
    self.dataObj=pickle.loads(_dataRecv)
TypeError: loads() argument 1 must be string, not None
[2019-10-01 14:08:39,973] : [INFO] : [bleServerLogger] : [bleServer.py] : [writeJsonFile] : text.json file successfully opened to <open file 'text.json', mode 'w' at 0x7fc21e678810>
[2019-10-01 14:08:39,974] : [INFO] : [bleServerLogger] : [bleServer.py] : [writeJsonFile] : Content dumped successfully to the text.json file
[2019-10-01 14:08:39,975] : [INFO] : [bleServerLogger] : [bleServer.py] : [writeJsonFile] : text.json file successfully closed
[2019-10-01 14:08:39,976] : [ERROR] : [bleServerLogger] : [bleServer.py] : [closeBluetoothSocket] : Failed to close the bluetooth sockets 
Traceback (most recent call last):
  File "bleServer.py", line 135, in closeBluetoothSocket
    self.clientSocket.close()
AttributeError: 'NoneType' object has no attribute 'close'
[2019-10-01 14:08:40,897] : [INFO] : [bleServerLogger] : [bleServer.py] : [getBluetoothSocket] : Bluetooth server socket successfully created for RFCOMM service...
[2019-10-01 14:08:40,897] : [INFO] : [bleServerLogger] : [bleServer.py] : [getBluetoothConnection] : Bluetooth server socket bind successfully on host  to PORT_ANY...
[2019-10-01 14:08:40,897] : [INFO] : [bleServerLogger] : [bleServer.py] : [getBluetoothConnection] : Bluetooth server socket put to listening mode successfully ...
[2019-10-01 14:08:40,898] : [INFO] : [bleServerLogger] : [bleServer.py] : [getBluetoothConnection] : Waiting for connection on RFCOMM channel 1
[2019-10-01 14:08:40,898] : [ERROR] : [bleServerLogger] : [bleServer.py] : [advertiseBluetoothService] : Failed to advertise bluetooth services  ... 
Traceback (most recent call last):
  File "bleServer.py", line 77, in advertiseBluetoothService
    profiles = [ SERIAL_PORT_PROFILE ],
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 176, in advertise_service
    raise BluetoothError (str (e))
BluetoothError: (2, 'No such file or directory')
[2019-10-01 14:09:53,037] : [ERROR] : [bleServerLogger] : [bleServer.py] : [acceptBluetoothConnection] : Failed to accept bluetooth connection ... 
Traceback (most recent call last):
  File "bleServer.py", line 86, in acceptBluetoothConnection
    self.clientSocket, clientInfo = self.serverSocket.accept()
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 129, in accept
    client, addr = self._sock.accept ()
KeyboardInterrupt
[2019-10-01 14:09:53,039] : [ERROR] : [bleServerLogger] : [bleServer.py] : [deserializedData] : Failed to de-serialized string ... 
Traceback (most recent call last):
  File "bleServer.py", line 113, in deserializedData
    self.dataObj=pickle.loads(_dataRecv)
TypeError: loads() argument 1 must be string, not None
[2019-10-01 14:09:53,041] : [INFO] : [bleServerLogger] : [bleServer.py] : [writeJsonFile] : text.json file successfully opened to <open file 'text.json', mode 'w' at 0x7fd0a5c6e810>
[2019-10-01 14:09:53,041] : [INFO] : [bleServerLogger] : [bleServer.py] : [writeJsonFile] : Content dumped successfully to the text.json file
[2019-10-01 14:09:53,041] : [INFO] : [bleServerLogger] : [bleServer.py] : [writeJsonFile] : text.json file successfully closed
[2019-10-01 14:09:53,041] : [ERROR] : [bleServerLogger] : [bleServer.py] : [closeBluetoothSocket] : Failed to close the bluetooth sockets 
Traceback (most recent call last):
  File "bleServer.py", line 135, in closeBluetoothSocket
    self.clientSocket.close()
AttributeError: 'NoneType' object has no attribute 'close'
[2019-10-01 14:09:56,444] : [INFO] : [bleServerLogger] : [bleServer.py] : [getBluetoothSocket] : Bluetooth server socket successfully created for RFCOMM service...
[2019-10-01 14:09:56,444] : [INFO] : [bleServerLogger] : [bleServer.py] : [getBluetoothConnection] : Bluetooth server socket bind successfully on host  to PORT_ANY...
[2019-10-01 14:09:56,444] : [INFO] : [bleServerLogger] : [bleServer.py] : [getBluetoothConnection] : Bluetooth server socket put to listening mode successfully ...
[2019-10-01 14:09:56,444] : [INFO] : [bleServerLogger] : [bleServer.py] : [getBluetoothConnection] : Waiting for connection on RFCOMM channel 1
[2019-10-01 14:09:56,444] : [ERROR] : [bleServerLogger] : [bleServer.py] : [advertiseBluetoothService] : Failed to advertise bluetooth services  ... 
Traceback (most recent call last):
  File "bleServer.py", line 77, in advertiseBluetoothService
    profiles = [ SERIAL_PORT_PROFILE ],
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 176, in advertise_service
    raise BluetoothError (str (e))
BluetoothError: (2, 'No such file or directory')
[2019-10-08 13:26:27,549] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-08 13:26:27,549] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-08 13:27:11,437] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Re-connecting  Bluetooth services : 2 attempt
[2019-10-08 13:27:15,304] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Found  Bluetooth services ..
[2019-10-08 13:27:15,305] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Protocol	: RFCOMM
[2019-10-08 13:27:15,306] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Name		: BluetoothServices
[2019-10-08 13:27:15,306] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Service-id	: 94F39D29-7D6D-437D-973B-FBA39E49D4EE
[2019-10-08 13:27:15,307] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Profiles	: [('1101', 256)]
[2019-10-08 13:27:15,308] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Service-class	: []
[2019-10-08 13:27:15,308] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Host		: B8:27:EB:B6:74:28
[2019-10-08 13:27:15,309] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Provider	: None
[2019-10-08 13:27:15,310] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Port		: 1
[2019-10-08 13:27:15,311] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Description	: None
[2019-10-08 13:27:15,312] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-08 13:27:15,312] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothConnection] : Connecting to "BluetoothServices" on B8:27:EB:B6:74:28 with port 1
[2019-10-08 13:27:15,313] : [ERROR] : [bleClientLogger] : [bleClient.py] : [getBluetoothConnection] : Failed to connect to "BluetoothServices" on address B8:27:EB:B6:74:28 with port 1
Traceback (most recent call last):
  File "bleClient.py", line 83, in getBluetoothConnection
    self.clientSocket.connect((bleServiceInfo['host'], bleServiceInfo['port']))
  File "<string>", line 5, in connect
BluetoothError: (16, 'Device or resource busy')
[2019-10-08 13:27:15,316] : [ERROR] : [bleClientLogger] : [bleClient.py] : [readJsonFile] : Failed to load content from the text.json
Traceback (most recent call last):
  File "bleClient.py", line 90, in readJsonFile
    jsonFileObj = open(self.lsonFile,"r")
AttributeError: bleClient instance has no attribute 'lsonFile'
[2019-10-08 13:27:15,318] : [INFO] : [bleClientLogger] : [bleClient.py] : [serializeData] : Object successfully converted to a serialized string
[2019-10-08 13:27:15,318] : [INFO] : [bleClientLogger] : [bleClient.py] : [sendData] : Sending data over bluetooth connection
[2019-10-08 13:27:15,319] : [ERROR] : [bleClientLogger] : [bleClient.py] : [sendData] : Failed to send data over bluetooth connection
Traceback (most recent call last):
  File "bleClient.py", line 110, in sendData
    self.clientSocket.send(_serializedData)
  File "<string>", line 5, in send
BluetoothError: (107, 'Transport endpoint is not connected')
[2019-10-08 13:27:15,321] : [INFO] : [bleClientLogger] : [bleClient.py] : [closeBluetoothSocket] : Bluetooth client socket successfully closed ...
[2019-10-08 13:27:28,368] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-08 13:27:28,368] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-08 13:27:53,156] : [ERROR] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Couldn't find the RaspberryPi Bluetooth service : Invalid uuid
Traceback (most recent call last):
  File "bleClient.py", line 51, in getBluetoothServices
    bleService = find_service( uuid = self.uuid, address = self.addr )
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 199, in find_service
    s.connect (addr)
KeyboardInterrupt
[2019-10-08 13:27:53,160] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-08 13:31:41,027] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-08 13:31:41,027] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-08 13:32:41,484] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Found  Bluetooth services ..
[2019-10-08 13:32:41,486] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Protocol	: RFCOMM
[2019-10-08 13:32:41,486] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Name		: BluetoothServices
[2019-10-08 13:32:41,487] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Service-id	: 94F39D29-7D6D-437D-973B-FBA39E49D4EE
[2019-10-08 13:32:41,488] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Profiles	: [('1101', 256)]
[2019-10-08 13:32:41,488] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Service-class	: []
[2019-10-08 13:32:41,489] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Host		: B8:27:EB:B6:74:28
[2019-10-08 13:32:41,489] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Provider	: None
[2019-10-08 13:32:41,490] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Port		: 1
[2019-10-08 13:32:41,490] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Description	: None
[2019-10-08 13:32:41,490] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-08 13:32:41,490] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothConnection] : Connecting to "BluetoothServices" on B8:27:EB:B6:74:28 with port 1
[2019-10-08 13:32:41,490] : [ERROR] : [bleClientLogger] : [bleClient.py] : [getBluetoothConnection] : Failed to connect to "BluetoothServices" on address B8:27:EB:B6:74:28 with port 1
Traceback (most recent call last):
  File "bleClient.py", line 83, in getBluetoothConnection
    self.clientSocket.connect((bleServiceInfo['host'], bleServiceInfo['port']))
  File "<string>", line 5, in connect
BluetoothError: (16, 'Device or resource busy')
[2019-10-08 13:32:41,491] : [INFO] : [bleClientLogger] : [bleClient.py] : [readJsonFile] : File successfully uploaded to <open file 'text.json', mode 'r' at 0x7ffb58e76780>
[2019-10-08 13:32:41,491] : [INFO] : [bleClientLogger] : [bleClient.py] : [readJsonFile] : Content loaded successfully from the text.json file
[2019-10-08 13:32:41,492] : [INFO] : [bleClientLogger] : [bleClient.py] : [serializeData] : Object successfully converted to a serialized string
[2019-10-08 13:32:41,492] : [INFO] : [bleClientLogger] : [bleClient.py] : [sendData] : Sending data over bluetooth connection
[2019-10-08 13:32:41,492] : [ERROR] : [bleClientLogger] : [bleClient.py] : [sendData] : Failed to send data over bluetooth connection
Traceback (most recent call last):
  File "bleClient.py", line 110, in sendData
    self.clientSocket.send(_serializedData)
  File "<string>", line 5, in send
BluetoothError: (107, 'Transport endpoint is not connected')
[2019-10-08 13:32:41,492] : [INFO] : [bleClientLogger] : [bleClient.py] : [closeBluetoothSocket] : Bluetooth client socket successfully closed ...
[2019-10-08 13:32:58,459] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-08 13:32:58,459] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-08 13:33:02,622] : [ERROR] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Couldn't find the RaspberryPi Bluetooth service : Invalid uuid
Traceback (most recent call last):
  File "bleClient.py", line 51, in getBluetoothServices
    bleService = find_service( uuid = self.uuid, address = self.addr )
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 199, in find_service
    s.connect (addr)
KeyboardInterrupt
[2019-10-08 13:33:02,626] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-08 13:33:08,750] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-08 13:33:08,750] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-08 13:34:09,830] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Found  Bluetooth services ..
[2019-10-08 13:34:09,831] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Protocol	: RFCOMM
[2019-10-08 13:34:09,832] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Name		: BluetoothServices
[2019-10-08 13:34:09,832] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Service-id	: 94F39D29-7D6D-437D-973B-FBA39E49D4EE
[2019-10-08 13:34:09,833] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Profiles	: [('1101', 256)]
[2019-10-08 13:34:09,834] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Service-class	: []
[2019-10-08 13:34:09,834] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Host		: B8:27:EB:B6:74:28
[2019-10-08 13:34:09,835] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Provider	: None
[2019-10-08 13:34:09,835] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Port		: 1
[2019-10-08 13:34:09,836] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Description	: None
[2019-10-08 13:34:09,837] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-08 13:34:09,838] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothConnection] : Connecting to "BluetoothServices" on B8:27:EB:B6:74:28 with port 1
[2019-10-08 13:34:13,469] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothConnection] : Connected successfully to BluetoothServices 
[2019-10-08 13:34:13,470] : [INFO] : [bleClientLogger] : [bleClient.py] : [readJsonFile] : File successfully uploaded to <open file 'text.json', mode 'r' at 0x7f19d0b25780>
[2019-10-08 13:34:13,471] : [INFO] : [bleClientLogger] : [bleClient.py] : [readJsonFile] : Content loaded successfully from the text.json file
[2019-10-08 13:34:13,471] : [INFO] : [bleClientLogger] : [bleClient.py] : [serializeData] : Object successfully converted to a serialized string
[2019-10-08 13:34:13,471] : [INFO] : [bleClientLogger] : [bleClient.py] : [sendData] : Sending data over bluetooth connection
[2019-10-08 13:34:13,972] : [INFO] : [bleClientLogger] : [bleClient.py] : [sendData] : Data sent successfully over bluetooth connection
[2019-10-08 13:34:13,973] : [INFO] : [bleClientLogger] : [bleClient.py] : [closeBluetoothSocket] : Bluetooth client socket successfully closed ...
[2019-10-08 13:38:08,761] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-08 13:38:08,761] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-08 13:38:19,124] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Re-connecting  Bluetooth services : 2 attempt
[2019-10-08 13:38:29,382] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Re-connecting  Bluetooth services : 3 attempt
[2019-10-08 13:38:29,383] : [ERROR] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Couldn't find the RaspberryPi Bluetooth service : Invalid uuid
Traceback (most recent call last):
  File "bleClient.py", line 56, in getBluetoothServices
    if not bleService: raise SystemExit(), KeyboardInterrupt()
TypeError: instance exception may not have a separate value
[2019-10-08 13:38:29,385] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-08 13:40:35,269] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-08 13:40:35,269] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-08 13:41:36,657] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Found  Bluetooth services ..
[2019-10-08 13:41:36,658] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Protocol	: RFCOMM
[2019-10-08 13:41:36,658] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Name		: BluetoothServices
[2019-10-08 13:41:36,658] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Service-id	: 94F39D29-7D6D-437D-973B-FBA39E49D4EE
[2019-10-08 13:41:36,659] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Profiles	: [('1101', 256)]
[2019-10-08 13:41:36,659] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Service-class	: []
[2019-10-08 13:41:36,660] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Host		: B8:27:EB:B6:74:28
[2019-10-08 13:41:36,660] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Provider	: None
[2019-10-08 13:41:36,660] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Port		: 1
[2019-10-08 13:41:36,661] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Description	: None
[2019-10-08 13:41:36,661] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-08 13:41:36,661] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothConnection] : Connecting to "BluetoothServices" on B8:27:EB:B6:74:28 with port 1
[2019-10-08 13:41:36,972] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothConnection] : Connected successfully to BluetoothServices 
[2019-10-08 13:41:36,973] : [INFO] : [bleClientLogger] : [bleClient.py] : [readJsonFile] : File successfully uploaded to <open file 'text.json', mode 'r' at 0x7ff635439780>
[2019-10-08 13:41:36,973] : [INFO] : [bleClientLogger] : [bleClient.py] : [readJsonFile] : Content loaded successfully from the text.json file
[2019-10-08 13:41:36,974] : [INFO] : [bleClientLogger] : [bleClient.py] : [serializeData] : Object successfully converted to a serialized string
[2019-10-08 13:41:36,974] : [INFO] : [bleClientLogger] : [bleClient.py] : [sendData] : Sending data over bluetooth connection
[2019-10-08 13:41:37,476] : [INFO] : [bleClientLogger] : [bleClient.py] : [sendData] : Data sent successfully over bluetooth connection
[2019-10-08 13:41:37,476] : [INFO] : [bleClientLogger] : [bleClient.py] : [closeBluetoothSocket] : Bluetooth client socket successfully closed ...
[2019-10-08 13:42:23,056] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-08 13:42:23,056] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-08 13:43:10,502] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Found  Bluetooth services ..
[2019-10-08 13:43:10,503] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Protocol	: RFCOMM
[2019-10-08 13:43:10,504] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Name		: BluetoothServices
[2019-10-08 13:43:10,505] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Service-id	: 94F39D29-7D6D-437D-973B-FBA39E49D4EE
[2019-10-08 13:43:10,506] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Profiles	: [('1101', 256)]
[2019-10-08 13:43:10,507] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Service-class	: []
[2019-10-08 13:43:10,507] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Host		: B8:27:EB:B6:74:28
[2019-10-08 13:43:10,508] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Provider	: None
[2019-10-08 13:43:10,509] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Port		: 1
[2019-10-08 13:43:10,510] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Description	: None
[2019-10-08 13:43:10,510] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-08 13:43:10,511] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothConnection] : Connecting to "BluetoothServices" on B8:27:EB:B6:74:28 with port 1
[2019-10-08 13:43:14,858] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothConnection] : Connected successfully to BluetoothServices 
[2019-10-08 13:43:14,858] : [INFO] : [bleClientLogger] : [bleClient.py] : [readJsonFile] : File successfully uploaded to <open file 'text.json', mode 'r' at 0x7f4db488e780>
[2019-10-08 13:43:14,858] : [INFO] : [bleClientLogger] : [bleClient.py] : [readJsonFile] : Content loaded successfully from the text.json file
[2019-10-08 13:43:14,858] : [INFO] : [bleClientLogger] : [bleClient.py] : [serializeData] : Object successfully converted to a serialized string
[2019-10-08 13:43:14,859] : [INFO] : [bleClientLogger] : [bleClient.py] : [sendData] : Sending data over bluetooth connection
[2019-10-08 13:43:15,360] : [INFO] : [bleClientLogger] : [bleClient.py] : [sendData] : Data sent successfully over bluetooth connection
[2019-10-08 13:43:15,361] : [INFO] : [bleClientLogger] : [bleClient.py] : [closeBluetoothSocket] : Bluetooth client socket successfully closed ...
[2019-10-08 13:47:23,895] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-08 13:47:23,895] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-08 13:48:04,927] : [ERROR] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Couldn't find the RaspberryPi Bluetooth service : Invalid uuid
Traceback (most recent call last):
  File "bleClient.py", line 51, in getBluetoothServices
    bleService = find_service( uuid = self.uuid, address = self.addr )
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 199, in find_service
    s.connect (addr)
KeyboardInterrupt
[2019-10-08 13:48:04,930] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-08 13:48:06,170] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-08 13:48:06,170] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-08 13:48:52,521] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Re-connecting  Bluetooth services : 2 attempt
[2019-10-08 13:49:42,726] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Found  Bluetooth services ..
[2019-10-08 13:49:42,727] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Protocol	: RFCOMM
[2019-10-08 13:49:42,728] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Name		: BluetoothServices
[2019-10-08 13:49:42,729] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Service-id	: 94F39D29-7D6D-437D-973B-FBA39E49D4EE
[2019-10-08 13:49:42,730] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Profiles	: [('1101', 256)]
[2019-10-08 13:49:42,731] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Service-class	: []
[2019-10-08 13:49:42,733] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Host		: B8:27:EB:B6:74:28
[2019-10-08 13:49:42,734] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Provider	: None
[2019-10-08 13:49:42,736] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Port		: 1
[2019-10-08 13:49:42,738] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Description	: None
[2019-10-08 13:49:42,745] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-08 13:49:42,746] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothConnection] : Connecting to "BluetoothServices" on B8:27:EB:B6:74:28 with port 1
[2019-10-08 13:49:42,944] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothConnection] : Connected successfully to BluetoothServices 
[2019-10-08 13:49:42,944] : [INFO] : [bleClientLogger] : [bleClient.py] : [readJsonFile] : File successfully uploaded to <open file 'text.json', mode 'r' at 0x7fe728554780>
[2019-10-08 13:49:42,944] : [INFO] : [bleClientLogger] : [bleClient.py] : [readJsonFile] : Content loaded successfully from the text.json file
[2019-10-08 13:49:42,944] : [INFO] : [bleClientLogger] : [bleClient.py] : [serializeData] : Object successfully converted to a serialized string
[2019-10-08 13:49:42,945] : [INFO] : [bleClientLogger] : [bleClient.py] : [sendData] : Sending data over bluetooth connection
[2019-10-08 13:49:43,445] : [INFO] : [bleClientLogger] : [bleClient.py] : [sendData] : Data sent successfully over bluetooth connection
[2019-10-08 13:49:43,446] : [INFO] : [bleClientLogger] : [bleClient.py] : [closeBluetoothSocket] : Bluetooth client socket successfully closed ...
[2019-10-08 14:08:11,660] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-08 14:08:11,660] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-08 14:09:06,804] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Found  Bluetooth services ..
[2019-10-08 14:09:06,805] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Protocol	: RFCOMM
[2019-10-08 14:09:06,806] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Name		: BluetoothServices
[2019-10-08 14:09:06,807] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Service-id	: 94F39D29-7D6D-437D-973B-FBA39E49D4EE
[2019-10-08 14:09:06,808] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Profiles	: [('1101', 256)]
[2019-10-08 14:09:06,809] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Service-class	: []
[2019-10-08 14:09:06,810] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Host		: B8:27:EB:B6:74:28
[2019-10-08 14:09:06,811] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Provider	: None
[2019-10-08 14:09:06,812] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Port		: 1
[2019-10-08 14:09:06,814] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Description	: None
[2019-10-08 14:09:06,815] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-08 14:09:06,816] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothConnection] : Connecting to "BluetoothServices" on B8:27:EB:B6:74:28 with port 1
[2019-10-08 14:09:06,945] : [ERROR] : [bleClientLogger] : [bleClient.py] : [getBluetoothConnection] : Failed to connect to "BluetoothServices" on address B8:27:EB:B6:74:28 with port 1
Traceback (most recent call last):
  File "bleClient.py", line 83, in getBluetoothConnection
    self.clientSocket.connect((bleServiceInfo['host'], bleServiceInfo['port']))
  File "<string>", line 5, in connect
BluetoothError: (111, 'Connection refused')
[2019-10-08 14:09:06,946] : [INFO] : [bleClientLogger] : [bleClient.py] : [readJsonFile] : File successfully uploaded to <open file 'text.json', mode 'r' at 0x7fb36ac78780>
[2019-10-08 14:09:06,946] : [INFO] : [bleClientLogger] : [bleClient.py] : [readJsonFile] : Content loaded successfully from the text.json file
[2019-10-08 14:09:06,946] : [INFO] : [bleClientLogger] : [bleClient.py] : [serializeData] : Object successfully converted to a serialized string
[2019-10-08 14:09:06,946] : [INFO] : [bleClientLogger] : [bleClient.py] : [sendData] : Sending data over bluetooth connection
[2019-10-08 14:09:06,946] : [ERROR] : [bleClientLogger] : [bleClient.py] : [sendData] : Failed to send data over bluetooth connection
Traceback (most recent call last):
  File "bleClient.py", line 110, in sendData
    self.clientSocket.send(_serializedData)
  File "<string>", line 5, in send
BluetoothError: (107, 'Transport endpoint is not connected')
[2019-10-08 14:09:06,946] : [INFO] : [bleClientLogger] : [bleClient.py] : [closeBluetoothSocket] : Bluetooth client socket successfully closed ...
[2019-10-08 14:09:52,383] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-08 14:09:52,384] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-08 14:10:00,397] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Re-connecting  Bluetooth services : 2 attempt
[2019-10-08 14:11:54,555] : [ERROR] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Couldn't find the RaspberryPi Bluetooth service : Invalid uuid
Traceback (most recent call last):
  File "bleClient.py", line 51, in getBluetoothServices
    bleService = find_service( uuid = self.uuid, address = self.addr )
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 199, in find_service
    s.connect (addr)
KeyboardInterrupt
[2019-10-08 14:11:54,559] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-08 14:12:00,072] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-08 14:12:00,072] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-08 14:13:13,968] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Re-connecting  Bluetooth services : 2 attempt
[2019-10-08 14:13:37,541] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Found  Bluetooth services ..
[2019-10-08 14:13:37,542] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Protocol	: RFCOMM
[2019-10-08 14:13:37,543] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Name		: BluetoothServices
[2019-10-08 14:13:37,544] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Service-id	: 94F39D29-7D6D-437D-973B-FBA39E49D4EE
[2019-10-08 14:13:37,544] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Profiles	: [('1101', 256)]
[2019-10-08 14:13:37,545] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Service-class	: []
[2019-10-08 14:13:37,546] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Host		: B8:27:EB:B6:74:28
[2019-10-08 14:13:37,547] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Provider	: None
[2019-10-08 14:13:37,547] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Port		: 1
[2019-10-08 14:13:37,548] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Description	: None
[2019-10-08 14:13:37,549] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-08 14:13:37,549] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothConnection] : Connecting to "BluetoothServices" on B8:27:EB:B6:74:28 with port 1
[2019-10-08 14:13:37,678] : [ERROR] : [bleClientLogger] : [bleClient.py] : [getBluetoothConnection] : Failed to connect to "BluetoothServices" on address B8:27:EB:B6:74:28 with port 1
Traceback (most recent call last):
  File "bleClient.py", line 83, in getBluetoothConnection
    self.clientSocket.connect((bleServiceInfo['host'], bleServiceInfo['port']))
  File "<string>", line 5, in connect
BluetoothError: (111, 'Connection refused')
[2019-10-08 14:13:37,681] : [INFO] : [bleClientLogger] : [bleClient.py] : [readJsonFile] : File successfully uploaded to <open file 'text.json', mode 'r' at 0x7f607e3be780>
[2019-10-08 14:13:37,682] : [INFO] : [bleClientLogger] : [bleClient.py] : [readJsonFile] : Content loaded successfully from the text.json file
[2019-10-08 14:13:37,683] : [INFO] : [bleClientLogger] : [bleClient.py] : [serializeData] : Object successfully converted to a serialized string
[2019-10-08 14:13:37,683] : [INFO] : [bleClientLogger] : [bleClient.py] : [sendData] : Sending data over bluetooth connection
[2019-10-08 14:13:37,684] : [ERROR] : [bleClientLogger] : [bleClient.py] : [sendData] : Failed to send data over bluetooth connection
Traceback (most recent call last):
  File "bleClient.py", line 110, in sendData
    self.clientSocket.send(_serializedData)
  File "<string>", line 5, in send
BluetoothError: (107, 'Transport endpoint is not connected')
[2019-10-08 14:13:37,685] : [INFO] : [bleClientLogger] : [bleClient.py] : [closeBluetoothSocket] : Bluetooth client socket successfully closed ...
[2019-10-08 14:28:28,232] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-08 14:28:28,232] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-08 14:29:06,526] : [ERROR] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Couldn't find the RaspberryPi Bluetooth service : Invalid uuid
Traceback (most recent call last):
  File "bleClient.py", line 51, in getBluetoothServices
    bleService = find_service( uuid = self.uuid, address = self.addr )
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 199, in find_service
    s.connect (addr)
KeyboardInterrupt
[2019-10-08 14:29:06,530] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-08 14:29:24,029] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-08 14:29:24,030] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-08 14:31:10,020] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Re-connecting  Bluetooth services : 2 attempt
[2019-10-08 14:32:48,625] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Re-connecting  Bluetooth services : 3 attempt
[2019-10-08 14:32:48,627] : [ERROR] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Couldn't find the RaspberryPi Bluetooth service : Invalid uuid
Traceback (most recent call last):
  File "bleClient.py", line 56, in getBluetoothServices
    if not bleService: raise SystemExit(), KeyboardInterrupt()
TypeError: instance exception may not have a separate value
[2019-10-08 14:32:48,630] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-08 14:33:56,915] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-08 14:33:56,916] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-08 14:35:35,337] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Re-connecting  Bluetooth services : 2 attempt
[2019-10-08 14:37:16,645] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Re-connecting  Bluetooth services : 3 attempt
[2019-10-08 14:37:16,646] : [ERROR] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Couldn't find the RaspberryPi Bluetooth service : Invalid uuid
Traceback (most recent call last):
  File "bleClient.py", line 56, in getBluetoothServices
    if not bleService: raise SystemExit(), KeyboardInterrupt()
TypeError: instance exception may not have a separate value
[2019-10-08 14:37:16,648] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-08 14:38:23,490] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-08 14:38:23,490] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-08 14:40:02,839] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Re-connecting  Bluetooth services : 2 attempt
[2019-10-08 14:41:40,804] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Re-connecting  Bluetooth services : 3 attempt
[2019-10-08 14:41:40,806] : [ERROR] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Couldn't find the RaspberryPi Bluetooth service : Invalid uuid
Traceback (most recent call last):
  File "bleClient.py", line 56, in getBluetoothServices
    if not bleService: raise SystemExit(), KeyboardInterrupt()
TypeError: instance exception may not have a separate value
[2019-10-08 14:41:40,808] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-08 14:42:51,884] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-08 14:42:51,885] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-08 14:44:28,740] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Re-connecting  Bluetooth services : 2 attempt
[2019-10-08 14:45:00,721] : [ERROR] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Couldn't find the RaspberryPi Bluetooth service : Invalid uuid
Traceback (most recent call last):
  File "bleClient.py", line 51, in getBluetoothServices
    bleService = find_service( uuid = self.uuid, address = self.addr )
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 199, in find_service
    s.connect (addr)
KeyboardInterrupt
[2019-10-08 14:45:00,724] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-08 14:47:16,689] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-08 14:47:16,689] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-08 14:48:54,791] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Re-connecting  Bluetooth services : 2 attempt
[2019-10-08 14:49:57,064] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Re-connecting  Bluetooth services : 3 attempt
[2019-10-08 14:49:57,066] : [ERROR] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Couldn't find the RaspberryPi Bluetooth service : Invalid uuid
Traceback (most recent call last):
  File "bleClient.py", line 56, in getBluetoothServices
    if not bleService: raise SystemExit(), KeyboardInterrupt()
TypeError: instance exception may not have a separate value
[2019-10-08 14:49:57,067] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-08 15:00:38,558] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-08 15:00:38,558] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-08 15:00:38,558] : [ERROR] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Couldn't find the RaspberryPi Bluetooth service : Invalid uuid
Traceback (most recent call last):
  File "bleClient.py", line 51, in getBluetoothServices
    bleService = find_service( uuid = self.uuid, address = self.addr )
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 186, in find_service
    devices = discover_devices ()
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 22, in discover_devices
    raise BluetoothError ("error communicating with local "
BluetoothError: error communicating with local bluetooth adapter
[2019-10-08 15:00:38,559] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-08 15:00:44,496] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-08 15:00:44,496] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-08 15:00:44,496] : [ERROR] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Couldn't find the RaspberryPi Bluetooth service : Invalid uuid
Traceback (most recent call last):
  File "bleClient.py", line 51, in getBluetoothServices
    bleService = find_service( uuid = self.uuid, address = self.addr )
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 186, in find_service
    devices = discover_devices ()
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 22, in discover_devices
    raise BluetoothError ("error communicating with local "
BluetoothError: error communicating with local bluetooth adapter
[2019-10-08 15:00:44,496] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-08 15:01:13,374] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-08 15:01:13,374] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-08 15:01:13,374] : [ERROR] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Couldn't find the RaspberryPi Bluetooth service : Invalid uuid
Traceback (most recent call last):
  File "bleClient.py", line 51, in getBluetoothServices
    bleService = find_service( uuid = self.uuid, address = self.addr )
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 186, in find_service
    devices = discover_devices ()
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 22, in discover_devices
    raise BluetoothError ("error communicating with local "
BluetoothError: error communicating with local bluetooth adapter
[2019-10-08 15:01:13,375] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-08 15:01:33,934] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-08 15:01:33,934] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-08 15:02:25,988] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Re-connecting  Bluetooth services : 2 attempt
[2019-10-08 15:03:18,751] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Re-connecting  Bluetooth services : 3 attempt
[2019-10-08 15:03:18,751] : [ERROR] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Couldn't find the RaspberryPi Bluetooth service : Invalid uuid
Traceback (most recent call last):
  File "bleClient.py", line 56, in getBluetoothServices
    if not bleService: raise SystemExit(), KeyboardInterrupt()
TypeError: instance exception may not have a separate value
[2019-10-08 15:03:18,752] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-08 15:03:50,584] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-08 15:03:50,584] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-08 15:04:23,220] : [ERROR] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Couldn't find the RaspberryPi Bluetooth service : Invalid uuid
Traceback (most recent call last):
  File "bleClient.py", line 51, in getBluetoothServices
    bleService = find_service( uuid = self.uuid, address = self.addr )
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 199, in find_service
    s.connect (addr)
KeyboardInterrupt
[2019-10-08 15:04:23,223] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-08 15:10:19,597] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-08 15:10:19,597] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-08 15:11:17,574] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Re-connecting  Bluetooth services : 2 attempt
[2019-10-08 15:12:22,608] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Re-connecting  Bluetooth services : 3 attempt
[2019-10-08 15:12:22,610] : [ERROR] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Couldn't find the RaspberryPi Bluetooth service : Invalid uuid
Traceback (most recent call last):
  File "bleClient.py", line 56, in getBluetoothServices
    if not bleService: raise SystemExit(), KeyboardInterrupt()
TypeError: instance exception may not have a separate value
[2019-10-08 15:12:22,613] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-08 15:22:12,978] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-08 15:22:12,978] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-08 15:23:10,177] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Re-connecting  Bluetooth services : 2 attempt
[2019-10-08 15:23:19,109] : [ERROR] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Couldn't find the RaspberryPi Bluetooth service : Invalid uuid
Traceback (most recent call last):
  File "bleClient.py", line 51, in getBluetoothServices
    bleService = find_service( uuid = self.uuid, address = self.addr )
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 186, in find_service
    devices = discover_devices ()
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 22, in discover_devices
    raise BluetoothError ("error communicating with local "
BluetoothError: error communicating with local bluetooth adapter
[2019-10-08 15:23:19,110] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-09 11:58:42,868] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-09 11:58:42,869] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-09 11:59:04,308] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Re-connecting  Bluetooth services : 2 attempt
[2019-10-09 11:59:25,313] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Re-connecting  Bluetooth services : 3 attempt
[2019-10-09 11:59:25,314] : [ERROR] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Couldn't find the RaspberryPi Bluetooth service : Invalid uuid
Traceback (most recent call last):
  File "bleClient.py", line 56, in getBluetoothServices
    if not bleService: raise SystemExit(), KeyboardInterrupt()
TypeError: instance exception may not have a separate value
[2019-10-09 11:59:25,319] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-09 12:33:56,598] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-09 12:33:56,598] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-09 12:34:16,413] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Re-connecting  Bluetooth services : 2 attempt
[2019-10-09 12:34:39,404] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Re-connecting  Bluetooth services : 3 attempt
[2019-10-09 12:34:39,405] : [ERROR] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Couldn't find the RaspberryPi Bluetooth service : Invalid uuid
Traceback (most recent call last):
  File "bleClient.py", line 56, in getBluetoothServices
    if not bleService: raise SystemExit(), KeyboardInterrupt()
TypeError: instance exception may not have a separate value
[2019-10-09 12:34:39,405] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-09 12:41:39,240] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-09 12:41:39,240] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-09 12:41:58,528] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Re-connecting  Bluetooth services : 2 attempt
[2019-10-09 12:42:17,427] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Re-connecting  Bluetooth services : 3 attempt
[2019-10-09 12:42:17,428] : [ERROR] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Couldn't find the RaspberryPi Bluetooth service : Invalid uuid
Traceback (most recent call last):
  File "bleClient.py", line 56, in getBluetoothServices
    if not bleService: raise SystemExit(), KeyboardInterrupt()
TypeError: instance exception may not have a separate value
[2019-10-09 12:42:17,431] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-09 12:42:56,996] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-09 12:42:56,997] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-09 12:43:05,336] : [ERROR] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Couldn't find the RaspberryPi Bluetooth service : Invalid uuid
Traceback (most recent call last):
  File "bleClient.py", line 51, in getBluetoothServices
    bleService = find_service( uuid = self.uuid, address = self.addr )
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 186, in find_service
    devices = discover_devices ()
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 19, in discover_devices
    results = _bt.hci_inquiry (sock, duration=duration, flush_cache=True)
KeyboardInterrupt
[2019-10-09 12:43:05,341] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-09 12:43:27,783] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-09 12:43:27,783] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-09 12:43:44,649] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Re-connecting  Bluetooth services : 2 attempt
[2019-10-09 12:44:02,049] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Re-connecting  Bluetooth services : 3 attempt
[2019-10-09 12:44:02,051] : [ERROR] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Couldn't find the RaspberryPi Bluetooth service : Invalid uuid
Traceback (most recent call last):
  File "bleClient.py", line 56, in getBluetoothServices
    if not bleService: raise SystemExit(), KeyboardInterrupt()
TypeError: instance exception may not have a separate value
[2019-10-09 12:44:02,055] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-09 12:53:29,050] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-09 12:53:29,050] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-09 12:53:47,520] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Re-connecting  Bluetooth services : 2 attempt
[2019-10-09 12:54:05,554] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Re-connecting  Bluetooth services : 3 attempt
[2019-10-09 12:54:05,555] : [ERROR] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Couldn't find the RaspberryPi Bluetooth service : Invalid uuid
Traceback (most recent call last):
  File "bleClient.py", line 56, in getBluetoothServices
    if not bleService: raise SystemExit(), KeyboardInterrupt()
TypeError: instance exception may not have a separate value
[2019-10-09 12:54:05,556] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-09 12:54:14,651] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-09 12:54:14,651] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-09 12:54:32,297] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Re-connecting  Bluetooth services : 2 attempt
[2019-10-09 12:54:50,247] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Re-connecting  Bluetooth services : 3 attempt
[2019-10-09 12:54:50,248] : [ERROR] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Couldn't find the RaspberryPi Bluetooth service : Invalid uuid
Traceback (most recent call last):
  File "bleClient.py", line 56, in getBluetoothServices
    if not bleService: raise SystemExit(), KeyboardInterrupt()
TypeError: instance exception may not have a separate value
[2019-10-09 12:54:50,250] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-09 12:58:11,586] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-09 12:58:11,587] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-09 12:58:28,840] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Re-connecting  Bluetooth services : 2 attempt
[2019-10-09 12:58:47,048] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Re-connecting  Bluetooth services : 3 attempt
[2019-10-09 12:58:47,049] : [ERROR] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Couldn't find the RaspberryPi Bluetooth service : Invalid uuid
Traceback (most recent call last):
  File "bleClient.py", line 56, in getBluetoothServices
    if not bleService: raise SystemExit(), KeyboardInterrupt()
TypeError: instance exception may not have a separate value
[2019-10-09 12:58:47,050] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-09 13:16:31,911] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-09 13:16:31,911] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-09 13:16:51,400] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Re-connecting  Bluetooth services : 2 attempt
[2019-10-09 13:17:10,859] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Re-connecting  Bluetooth services : 3 attempt
[2019-10-09 13:17:10,860] : [ERROR] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Couldn't find the RaspberryPi Bluetooth service : Invalid uuid
Traceback (most recent call last):
  File "bleClient.py", line 56, in getBluetoothServices
    if not bleService: raise SystemExit(), KeyboardInterrupt()
TypeError: instance exception may not have a separate value
[2019-10-09 13:17:10,862] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-09 13:17:23,552] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-09 13:17:23,552] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-09 13:17:46,923] : [ERROR] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Couldn't find the RaspberryPi Bluetooth service : Invalid uuid
Traceback (most recent call last):
  File "bleClient.py", line 51, in getBluetoothServices
    bleService = find_service( uuid = self.uuid, address = self.addr )
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 199, in find_service
    s.connect (addr)
KeyboardInterrupt
[2019-10-09 13:17:46,926] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-09 13:20:59,494] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-09 13:20:59,494] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-09 13:21:18,915] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Re-connecting  Bluetooth services : 2 attempt
[2019-10-09 13:21:39,263] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Re-connecting  Bluetooth services : 3 attempt
[2019-10-09 13:21:39,265] : [ERROR] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Couldn't find the RaspberryPi Bluetooth service : Invalid uuid
Traceback (most recent call last):
  File "bleClient.py", line 56, in getBluetoothServices
    if not bleService: raise SystemExit(), KeyboardInterrupt()
TypeError: instance exception may not have a separate value
[2019-10-09 13:21:39,267] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-09 13:23:18,185] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-09 13:23:18,186] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-09 13:23:33,164] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Found  Bluetooth services ..
[2019-10-09 13:23:33,165] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Protocol	: RFCOMM
[2019-10-09 13:23:33,166] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Name		: BluetoothServices
[2019-10-09 13:23:33,167] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Service-id	: 94F39D29-7D6D-437D-973B-FBA39E49D4EE
[2019-10-09 13:23:33,168] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Profiles	: [('1101', 256)]
[2019-10-09 13:23:33,169] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Service-class	: []
[2019-10-09 13:23:33,170] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Host		: B8:27:EB:B6:74:28
[2019-10-09 13:23:33,170] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Provider	: None
[2019-10-09 13:23:33,171] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Port		: 1
[2019-10-09 13:23:33,173] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Description	: None
[2019-10-09 13:23:33,173] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-09 13:23:33,174] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothConnection] : Connecting to "BluetoothServices" on B8:27:EB:B6:74:28 with port 1
[2019-10-09 13:23:34,087] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothConnection] : Connected successfully to BluetoothServices 
[2019-10-09 13:23:34,089] : [INFO] : [bleClientLogger] : [bleClient.py] : [readJsonFile] : File successfully uploaded to <open file 'text.json', mode 'r' at 0x7fb41285b780>
[2019-10-09 13:23:34,090] : [INFO] : [bleClientLogger] : [bleClient.py] : [readJsonFile] : Content loaded successfully from the text.json file
[2019-10-09 13:23:34,091] : [INFO] : [bleClientLogger] : [bleClient.py] : [serializeData] : Object successfully converted to a serialized string
[2019-10-09 13:23:34,091] : [INFO] : [bleClientLogger] : [bleClient.py] : [sendData] : Sending data over bluetooth connection
[2019-10-09 13:23:34,593] : [INFO] : [bleClientLogger] : [bleClient.py] : [sendData] : Data sent successfully over bluetooth connection
[2019-10-09 13:23:34,593] : [INFO] : [bleClientLogger] : [bleClient.py] : [closeBluetoothSocket] : Bluetooth client socket successfully closed ...
[2019-10-09 13:25:34,053] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-09 13:25:34,053] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-09 13:26:08,790] : [ERROR] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Couldn't find the RaspberryPi Bluetooth service : Invalid uuid
Traceback (most recent call last):
  File "bleClient.py", line 51, in getBluetoothServices
    bleService = find_service( uuid = self.uuid, address = self.addr )
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 199, in find_service
    s.connect (addr)
KeyboardInterrupt
[2019-10-09 13:26:08,794] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-09 13:26:18,035] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-09 13:26:18,035] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-09 13:26:33,847] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Found  Bluetooth services ..
[2019-10-09 13:26:33,848] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Protocol	: RFCOMM
[2019-10-09 13:26:33,848] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Name		: BluetoothServices
[2019-10-09 13:26:33,849] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Service-id	: 94F39D29-7D6D-437D-973B-FBA39E49D4EE
[2019-10-09 13:26:33,849] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Profiles	: [('1101', 256)]
[2019-10-09 13:26:33,850] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Service-class	: []
[2019-10-09 13:26:33,850] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Host		: B8:27:EB:B6:74:28
[2019-10-09 13:26:33,851] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Provider	: None
[2019-10-09 13:26:33,852] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Port		: 1
[2019-10-09 13:26:33,852] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Description	: None
[2019-10-09 13:26:33,853] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-09 13:26:33,853] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothConnection] : Connecting to "BluetoothServices" on B8:27:EB:B6:74:28 with port 1
[2019-10-09 13:26:34,702] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothConnection] : Connected successfully to BluetoothServices 
[2019-10-09 13:26:34,702] : [INFO] : [bleClientLogger] : [bleClient.py] : [readJsonFile] : File successfully uploaded to <open file 'text.json', mode 'r' at 0x7fa64da0f780>
[2019-10-09 13:26:34,702] : [INFO] : [bleClientLogger] : [bleClient.py] : [readJsonFile] : Content loaded successfully from the text.json file
[2019-10-09 13:26:34,703] : [INFO] : [bleClientLogger] : [bleClient.py] : [serializeData] : Object successfully converted to a serialized string
[2019-10-09 13:26:34,703] : [INFO] : [bleClientLogger] : [bleClient.py] : [sendData] : Sending data over bluetooth connection
[2019-10-09 13:26:35,204] : [INFO] : [bleClientLogger] : [bleClient.py] : [sendData] : Data sent successfully over bluetooth connection
[2019-10-09 13:26:35,206] : [INFO] : [bleClientLogger] : [bleClient.py] : [closeBluetoothSocket] : Bluetooth client socket successfully closed ...
[2019-10-09 15:07:21,641] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-09 15:07:21,642] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-09 15:07:37,102] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Re-connecting  Bluetooth services : 2 attempt
[2019-10-09 15:08:03,921] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Re-connecting  Bluetooth services : 3 attempt
[2019-10-09 15:08:03,922] : [ERROR] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Couldn't find the RaspberryPi Bluetooth service : Invalid uuid
Traceback (most recent call last):
  File "bleClient.py", line 56, in getBluetoothServices
    if not bleService: raise SystemExit(), KeyboardInterrupt()
TypeError: instance exception may not have a separate value
[2019-10-09 15:08:03,923] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-09 15:08:06,371] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-09 15:08:06,372] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-09 15:08:27,012] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Found  Bluetooth services ..
[2019-10-09 15:08:27,013] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Protocol	: RFCOMM
[2019-10-09 15:08:27,013] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Name		: BluetoothServices
[2019-10-09 15:08:27,014] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Service-id	: 94F39D29-7D6D-437D-973B-FBA39E49D4EE
[2019-10-09 15:08:27,014] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Profiles	: [('1101', 256)]
[2019-10-09 15:08:27,014] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Service-class	: []
[2019-10-09 15:08:27,015] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Host		: B8:27:EB:B6:74:28
[2019-10-09 15:08:27,015] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Provider	: None
[2019-10-09 15:08:27,015] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Port		: 1
[2019-10-09 15:08:27,016] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Description	: None
[2019-10-09 15:08:27,016] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-09 15:08:27,017] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothConnection] : Connecting to "BluetoothServices" on B8:27:EB:B6:74:28 with port 1
[2019-10-09 15:08:31,896] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothConnection] : Connected successfully to BluetoothServices 
[2019-10-09 15:08:31,897] : [INFO] : [bleClientLogger] : [bleClient.py] : [readJsonFile] : File successfully uploaded to <open file 'text.json', mode 'r' at 0x7f5216d9e780>
[2019-10-09 15:08:31,898] : [INFO] : [bleClientLogger] : [bleClient.py] : [readJsonFile] : Content loaded successfully from the text.json file
[2019-10-09 15:08:31,899] : [INFO] : [bleClientLogger] : [bleClient.py] : [serializeData] : Object successfully converted to a serialized string
[2019-10-09 15:08:31,900] : [INFO] : [bleClientLogger] : [bleClient.py] : [sendData] : Sending data over bluetooth connection
[2019-10-09 15:08:32,401] : [INFO] : [bleClientLogger] : [bleClient.py] : [sendData] : Data sent successfully over bluetooth connection
[2019-10-09 15:08:32,402] : [INFO] : [bleClientLogger] : [bleClient.py] : [closeBluetoothSocket] : Bluetooth client socket successfully closed ...
[2019-10-09 15:11:59,474] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-09 15:11:59,474] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-09 15:12:07,201] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Re-connecting  Bluetooth services : 2 attempt
[2019-10-09 15:12:26,389] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Found  Bluetooth services ..
[2019-10-09 15:12:26,390] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Protocol	: RFCOMM
[2019-10-09 15:12:26,391] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Name		: BluetoothServices
[2019-10-09 15:12:26,391] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Service-id	: 94F39D29-7D6D-437D-973B-FBA39E49D4EE
[2019-10-09 15:12:26,392] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Profiles	: [('1101', 256)]
[2019-10-09 15:12:26,393] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Service-class	: []
[2019-10-09 15:12:26,394] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Host		: B8:27:EB:B6:74:28
[2019-10-09 15:12:26,395] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Provider	: None
[2019-10-09 15:12:26,396] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Port		: 1
[2019-10-09 15:12:26,396] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Description	: None
[2019-10-09 15:12:26,397] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-09 15:12:26,398] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothConnection] : Connecting to "BluetoothServices" on B8:27:EB:B6:74:28 with port 1
[2019-10-09 15:12:27,119] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothConnection] : Connected successfully to BluetoothServices 
[2019-10-09 15:12:27,120] : [INFO] : [bleClientLogger] : [bleClient.py] : [readJsonFile] : File successfully uploaded to <open file 'text.json', mode 'r' at 0x7f8dc27e8780>
[2019-10-09 15:12:27,121] : [INFO] : [bleClientLogger] : [bleClient.py] : [readJsonFile] : Content loaded successfully from the text.json file
[2019-10-09 15:12:27,121] : [INFO] : [bleClientLogger] : [bleClient.py] : [serializeData] : Object successfully converted to a serialized string
[2019-10-09 15:12:27,122] : [INFO] : [bleClientLogger] : [bleClient.py] : [sendData] : Sending data over bluetooth connection
[2019-10-09 15:12:27,624] : [INFO] : [bleClientLogger] : [bleClient.py] : [sendData] : Data sent successfully over bluetooth connection
[2019-10-09 15:12:27,625] : [INFO] : [bleClientLogger] : [bleClient.py] : [closeBluetoothSocket] : Bluetooth client socket successfully closed ...
[2019-10-15 11:46:53,180] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-15 11:46:53,181] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-15 11:46:58,620] : [ERROR] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Couldn't find the RaspberryPi Bluetooth service : Invalid uuid
Traceback (most recent call last):
  File "bleClient.py", line 51, in getBluetoothServices
    bleService = find_service( uuid = self.uuid, address = self.addr )
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 186, in find_service
    devices = discover_devices ()
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 19, in discover_devices
    results = _bt.hci_inquiry (sock, duration=duration, flush_cache=True)
KeyboardInterrupt
[2019-10-15 11:46:58,627] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-15 11:50:00,518] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-15 11:50:00,518] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-15 11:50:29,491] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Found  Bluetooth services ..
[2019-10-15 11:50:29,493] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Protocol	: RFCOMM
[2019-10-15 11:50:29,493] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Name		: BluetoothServices
[2019-10-15 11:50:29,494] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Service-id	: 94F39D29-7D6D-437D-973B-FBA39E49D4EE
[2019-10-15 11:50:29,495] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Profiles	: [('1101', 256)]
[2019-10-15 11:50:29,496] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Service-class	: []
[2019-10-15 11:50:29,496] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Host		: B8:27:EB:B6:74:28
[2019-10-15 11:50:29,497] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Provider	: None
[2019-10-15 11:50:29,498] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Port		: 1
[2019-10-15 11:50:29,499] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Description	: None
[2019-10-15 11:50:29,499] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-15 11:50:29,500] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothConnection] : Connecting to "BluetoothServices" on B8:27:EB:B6:74:28 with port 1
[2019-10-15 11:50:34,523] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothConnection] : Connected successfully to BluetoothServices 
[2019-10-15 11:50:34,524] : [INFO] : [bleClientLogger] : [bleClient.py] : [readJsonFile] : File successfully uploaded to <open file 'text.json', mode 'r' at 0x7fc065ebc780>
[2019-10-15 11:50:34,526] : [INFO] : [bleClientLogger] : [bleClient.py] : [readJsonFile] : Content loaded successfully from the text.json file
[2019-10-15 11:50:34,527] : [INFO] : [bleClientLogger] : [bleClient.py] : [serializeData] : Object successfully converted to a serialized string
[2019-10-15 11:50:34,527] : [INFO] : [bleClientLogger] : [bleClient.py] : [sendData] : Sending data over bluetooth connection
[2019-10-15 11:50:35,028] : [INFO] : [bleClientLogger] : [bleClient.py] : [sendData] : Data sent successfully over bluetooth connection
[2019-10-15 11:50:35,030] : [INFO] : [bleClientLogger] : [bleClient.py] : [closeBluetoothSocket] : Bluetooth client socket successfully closed ...
[2019-10-15 12:24:59,560] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-15 12:24:59,560] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-15 12:25:26,707] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Re-connecting  Bluetooth services : 2 attempt
[2019-10-15 12:25:30,344] : [ERROR] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Couldn't find the RaspberryPi Bluetooth service : Invalid uuid
Traceback (most recent call last):
  File "bleClient.py", line 51, in getBluetoothServices
    bleService = find_service( uuid = self.uuid, address = self.addr )
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 186, in find_service
    devices = discover_devices ()
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 19, in discover_devices
    results = _bt.hci_inquiry (sock, duration=duration, flush_cache=True)
KeyboardInterrupt
[2019-10-15 12:25:30,347] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-15 12:26:54,706] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-15 12:26:54,706] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-15 12:27:30,974] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Found  Bluetooth services ..
[2019-10-15 12:27:30,974] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Protocol	: RFCOMM
[2019-10-15 12:27:30,974] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Name		: BluetoothServices
[2019-10-15 12:27:30,974] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Service-id	: 94F39D29-7D6D-437D-973B-FBA39E49D4EE
[2019-10-15 12:27:30,975] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Profiles	: [('1101', 256)]
[2019-10-15 12:27:30,975] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Service-class	: []
[2019-10-15 12:27:30,975] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Host		: B8:27:EB:B6:74:28
[2019-10-15 12:27:30,975] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Provider	: None
[2019-10-15 12:27:30,976] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Port		: 1
[2019-10-15 12:27:30,976] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Description	: None
[2019-10-15 12:27:30,976] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-15 12:27:30,976] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothConnection] : Connecting to "BluetoothServices" on B8:27:EB:B6:74:28 with port 1
[2019-10-15 12:27:35,444] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothConnection] : Connected successfully to BluetoothServices 
[2019-10-15 12:27:35,445] : [INFO] : [bleClientLogger] : [bleClient.py] : [readJsonFile] : File successfully uploaded to <open file 'text.json', mode 'r' at 0x7f316f8c8780>
[2019-10-15 12:27:35,446] : [INFO] : [bleClientLogger] : [bleClient.py] : [readJsonFile] : Content loaded successfully from the text.json file
[2019-10-15 12:27:35,447] : [INFO] : [bleClientLogger] : [bleClient.py] : [serializeData] : Object successfully converted to a serialized string
[2019-10-15 12:27:35,448] : [INFO] : [bleClientLogger] : [bleClient.py] : [sendData] : Sending data over bluetooth connection
[2019-10-15 12:27:35,950] : [INFO] : [bleClientLogger] : [bleClient.py] : [sendData] : Data sent successfully over bluetooth connection
[2019-10-15 12:27:35,950] : [INFO] : [bleClientLogger] : [bleClient.py] : [closeBluetoothSocket] : Bluetooth client socket successfully closed ...
[2019-10-15 12:33:55,372] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-15 12:33:55,372] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-15 12:33:59,639] : [ERROR] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Couldn't find the RaspberryPi Bluetooth service : Invalid uuid
Traceback (most recent call last):
  File "bleClient.py", line 51, in getBluetoothServices
    bleService = find_service( uuid = self.uuid, address = self.addr )
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 186, in find_service
    devices = discover_devices ()
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 19, in discover_devices
    results = _bt.hci_inquiry (sock, duration=duration, flush_cache=True)
KeyboardInterrupt
[2019-10-15 12:33:59,643] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-15 13:03:08,071] : [INFO] : [bleServerLogger] : [bleServer.py] : [getBluetoothSocket] : Bluetooth server socket successfully created for RFCOMM service...
[2019-10-15 13:03:08,071] : [INFO] : [bleServerLogger] : [bleServer.py] : [getBluetoothConnection] : Bluetooth server socket bind successfully on host  to PORT_ANY...
[2019-10-15 13:03:08,072] : [INFO] : [bleServerLogger] : [bleServer.py] : [getBluetoothConnection] : Bluetooth server socket put to listening mode successfully ...
[2019-10-15 13:03:08,072] : [INFO] : [bleServerLogger] : [bleServer.py] : [getBluetoothConnection] : Waiting for connection on RFCOMM channel 1
[2019-10-15 13:03:08,072] : [ERROR] : [bleServerLogger] : [bleServer.py] : [advertiseBluetoothService] : Failed to advertise bluetooth services  ... 
Traceback (most recent call last):
  File "bleServer.py", line 77, in advertiseBluetoothService
    profiles = [ SERIAL_PORT_PROFILE ],
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 176, in advertise_service
    raise BluetoothError (str (e))
BluetoothError: (2, 'No such file or directory')
[2019-10-15 13:03:14,991] : [ERROR] : [bleServerLogger] : [bleServer.py] : [acceptBluetoothConnection] : Failed to accept bluetooth connection ... 
Traceback (most recent call last):
  File "bleServer.py", line 86, in acceptBluetoothConnection
    self.clientSocket, clientInfo = self.serverSocket.accept()
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 129, in accept
    client, addr = self._sock.accept ()
KeyboardInterrupt
[2019-10-15 13:03:14,993] : [ERROR] : [bleServerLogger] : [bleServer.py] : [deserializedData] : Failed to de-serialized string ... 
Traceback (most recent call last):
  File "bleServer.py", line 113, in deserializedData
    self.dataObj=pickle.loads(_dataRecv)
TypeError: loads() argument 1 must be string, not None
[2019-10-15 13:03:14,995] : [INFO] : [bleServerLogger] : [bleServer.py] : [writeJsonFile] : text.json file successfully opened to <open file 'text.json', mode 'w' at 0x7f8497e5f810>
[2019-10-15 13:03:14,996] : [INFO] : [bleServerLogger] : [bleServer.py] : [writeJsonFile] : Content dumped successfully to the text.json file
[2019-10-15 13:03:14,997] : [INFO] : [bleServerLogger] : [bleServer.py] : [writeJsonFile] : text.json file successfully closed
[2019-10-15 13:03:14,997] : [ERROR] : [bleServerLogger] : [bleServer.py] : [closeBluetoothSocket] : Failed to close the bluetooth sockets 
Traceback (most recent call last):
  File "bleServer.py", line 135, in closeBluetoothSocket
    self.clientSocket.close()
AttributeError: 'NoneType' object has no attribute 'close'
[2019-10-15 13:03:21,293] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-15 13:03:21,293] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-15 13:03:24,146] : [ERROR] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Couldn't find the RaspberryPi Bluetooth service : Invalid uuid
Traceback (most recent call last):
  File "bleClient.py", line 51, in getBluetoothServices
    bleService = find_service( uuid = self.uuid, address = self.addr )
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 186, in find_service
    devices = discover_devices ()
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 19, in discover_devices
    results = _bt.hci_inquiry (sock, duration=duration, flush_cache=True)
KeyboardInterrupt
[2019-10-15 13:03:24,151] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-15 13:03:44,306] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-15 13:03:44,306] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-15 13:03:47,872] : [ERROR] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Couldn't find the RaspberryPi Bluetooth service : Invalid uuid
Traceback (most recent call last):
  File "bleClient.py", line 51, in getBluetoothServices
    bleService = find_service( uuid = self.uuid, address = self.addr )
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 186, in find_service
    devices = discover_devices ()
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 19, in discover_devices
    results = _bt.hci_inquiry (sock, duration=duration, flush_cache=True)
KeyboardInterrupt
[2019-10-15 13:03:47,877] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-15 13:03:56,030] : [INFO] : [bleClientLogger] : [bleClient.py] : [<module>] : Setup logging configuration
[2019-10-15 13:03:56,031] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-15 13:03:59,257] : [ERROR] : [bleClientLogger] : [bleClient.py] : [getBluetoothServices] : Couldn't find the RaspberryPi Bluetooth service : Invalid uuid
Traceback (most recent call last):
  File "bleClient.py", line 51, in getBluetoothServices
    bleService = find_service( uuid = self.uuid, address = self.addr )
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 186, in find_service
    devices = discover_devices ()
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 19, in discover_devices
    results = _bt.hci_inquiry (sock, duration=duration, flush_cache=True)
KeyboardInterrupt
[2019-10-15 13:03:59,262] : [INFO] : [bleClientLogger] : [bleClient.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-15 14:51:51,785] : [INFO] : [bleClientLogger] : [bleSender.py] : [<module>] : Setup logging configuration
[2019-10-15 14:51:51,785] : [INFO] : [bleClientLogger] : [bleSender.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-15 14:52:24,861] : [INFO] : [bleClientLogger] : [bleSender.py] : [getBluetoothServices] : Found  Bluetooth services ..
[2019-10-15 14:52:24,863] : [INFO] : [bleClientLogger] : [bleSender.py] : [getBluetoothServices] : Protocol	: RFCOMM
[2019-10-15 14:52:24,865] : [INFO] : [bleClientLogger] : [bleSender.py] : [getBluetoothServices] : Name		: BluetoothServices
[2019-10-15 14:52:24,866] : [INFO] : [bleClientLogger] : [bleSender.py] : [getBluetoothServices] : Service-id	: 94F39D29-7D6D-437D-973B-FBA39E49D4EE
[2019-10-15 14:52:24,866] : [INFO] : [bleClientLogger] : [bleSender.py] : [getBluetoothServices] : Profiles	: [('1101', 256)]
[2019-10-15 14:52:24,867] : [INFO] : [bleClientLogger] : [bleSender.py] : [getBluetoothServices] : Service-class	: []
[2019-10-15 14:52:24,868] : [INFO] : [bleClientLogger] : [bleSender.py] : [getBluetoothServices] : Host		: B8:27:EB:B6:74:28
[2019-10-15 14:52:24,869] : [INFO] : [bleClientLogger] : [bleSender.py] : [getBluetoothServices] : Provider	: None
[2019-10-15 14:52:24,869] : [INFO] : [bleClientLogger] : [bleSender.py] : [getBluetoothServices] : Port		: 1
[2019-10-15 14:52:24,870] : [INFO] : [bleClientLogger] : [bleSender.py] : [getBluetoothServices] : Description	: None
[2019-10-15 14:52:24,871] : [INFO] : [bleClientLogger] : [bleSender.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-15 14:52:24,872] : [INFO] : [bleClientLogger] : [bleSender.py] : [getBluetoothConnection] : Connecting to "BluetoothServices" on B8:27:EB:B6:74:28 with port 1
[2019-10-15 14:52:26,701] : [INFO] : [bleClientLogger] : [bleSender.py] : [getBluetoothConnection] : Connected successfully to BluetoothServices 
[2019-10-15 14:52:26,702] : [INFO] : [bleClientLogger] : [bleSender.py] : [readJsonFile] : File successfully uploaded to <open file 'text.json', mode 'r' at 0x7f0047238780>
[2019-10-15 14:52:26,702] : [INFO] : [bleClientLogger] : [bleSender.py] : [readJsonFile] : Content loaded successfully from the text.json file
[2019-10-15 14:52:26,703] : [INFO] : [bleClientLogger] : [bleSender.py] : [serializeData] : Object successfully converted to a serialized string
[2019-10-15 14:52:26,703] : [INFO] : [bleClientLogger] : [bleSender.py] : [sendData] : Sending data over bluetooth connection
[2019-10-15 14:52:27,204] : [INFO] : [bleClientLogger] : [bleSender.py] : [sendData] : Data sent successfully over bluetooth connection
[2019-10-15 14:52:27,205] : [INFO] : [bleClientLogger] : [bleSender.py] : [closeBluetoothSocket] : Bluetooth client socket successfully closed ...
[2019-10-15 14:57:00,052] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothSocket] : Bluetooth server socket successfully created for RFCOMM service...
[2019-10-15 14:57:00,052] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket bind successfully on host  to PORT_ANY...
[2019-10-15 14:57:00,052] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket put to listening mode successfully ...
[2019-10-15 14:57:00,052] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Waiting for connection on RFCOMM channel 1
[2019-10-15 14:57:00,052] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [advertiseBluetoothService] : Failed to advertise bluetooth services  ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 77, in advertiseBluetoothService
    profiles = [ SERIAL_PORT_PROFILE ],
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 176, in advertise_service
    raise BluetoothError (str (e))
BluetoothError: (2, 'No such file or directory')
[2019-10-15 14:57:26,593] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [acceptBluetoothConnection] : Failed to accept bluetooth connection ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 86, in acceptBluetoothConnection
    self.clientSocket, clientInfo = self.serverSocket.accept()
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 129, in accept
    client, addr = self._sock.accept ()
KeyboardInterrupt
[2019-10-15 14:57:26,595] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [deserializedData] : Failed to de-serialized string ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 113, in deserializedData
    self.dataObj=pickle.loads(_dataRecv)
TypeError: loads() argument 1 must be string, not None
[2019-10-15 14:57:26,596] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : text.json file successfully opened to <open file 'text.json', mode 'w' at 0x7f47d7d49810>
[2019-10-15 14:57:26,597] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : Content dumped successfully to the text.json file
[2019-10-15 14:57:26,598] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : text.json file successfully closed
[2019-10-15 14:57:26,598] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [closeBluetoothSocket] : Failed to close the bluetooth sockets 
Traceback (most recent call last):
  File "bleReceiver.py", line 135, in closeBluetoothSocket
    self.clientSocket.close()
AttributeError: 'NoneType' object has no attribute 'close'
[2019-10-15 15:04:30,213] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothSocket] : Bluetooth server socket successfully created for RFCOMM service...
[2019-10-15 15:04:30,213] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket bind successfully on host  to PORT_ANY...
[2019-10-15 15:04:30,214] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket put to listening mode successfully ...
[2019-10-15 15:04:30,214] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Waiting for connection on RFCOMM channel 1
[2019-10-15 15:04:30,214] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [advertiseBluetoothService] : Failed to advertise bluetooth services  ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 77, in advertiseBluetoothService
    profiles = [ SERIAL_PORT_PROFILE ],
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 176, in advertise_service
    raise BluetoothError (str (e))
BluetoothError: (2, 'No such file or directory')
[2019-10-15 15:07:39,784] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [acceptBluetoothConnection] : Failed to accept bluetooth connection ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 86, in acceptBluetoothConnection
    self.clientSocket, clientInfo = self.serverSocket.accept()
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 129, in accept
    client, addr = self._sock.accept ()
KeyboardInterrupt
[2019-10-15 15:07:39,787] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [deserializedData] : Failed to de-serialized string ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 113, in deserializedData
    self.dataObj=pickle.loads(_dataRecv)
TypeError: loads() argument 1 must be string, not None
[2019-10-15 15:07:39,789] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : text.json file successfully opened to <open file 'text.json', mode 'w' at 0x7f27930b7810>
[2019-10-15 15:07:39,790] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : Content dumped successfully to the text.json file
[2019-10-15 15:07:39,791] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : text.json file successfully closed
[2019-10-15 15:07:39,791] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [closeBluetoothSocket] : Failed to close the bluetooth sockets 
Traceback (most recent call last):
  File "bleReceiver.py", line 135, in closeBluetoothSocket
    self.clientSocket.close()
AttributeError: 'NoneType' object has no attribute 'close'
[2019-10-15 15:08:00,991] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothSocket] : Bluetooth server socket successfully created for RFCOMM service...
[2019-10-15 15:08:00,991] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket bind successfully on host  to PORT_ANY...
[2019-10-15 15:08:00,992] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket put to listening mode successfully ...
[2019-10-15 15:08:00,992] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Waiting for connection on RFCOMM channel 1
[2019-10-15 15:08:00,992] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [advertiseBluetoothService] : Failed to advertise bluetooth services  ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 77, in advertiseBluetoothService
    profiles = [ SERIAL_PORT_PROFILE ],
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 176, in advertise_service
    raise BluetoothError (str (e))
BluetoothError: (2, 'No such file or directory')
[2019-10-15 15:21:41,343] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [acceptBluetoothConnection] : Failed to accept bluetooth connection ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 86, in acceptBluetoothConnection
    self.clientSocket, clientInfo = self.serverSocket.accept()
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 129, in accept
    client, addr = self._sock.accept ()
KeyboardInterrupt
[2019-10-15 15:21:41,346] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [deserializedData] : Failed to de-serialized string ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 113, in deserializedData
    self.dataObj=pickle.loads(_dataRecv)
TypeError: loads() argument 1 must be string, not None
[2019-10-15 15:21:41,348] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : text.json file successfully opened to <open file 'text.json', mode 'w' at 0x7f4059a2c810>
[2019-10-15 15:21:41,349] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : Content dumped successfully to the text.json file
[2019-10-15 15:21:41,350] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : text.json file successfully closed
[2019-10-15 15:21:41,351] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [closeBluetoothSocket] : Failed to close the bluetooth sockets 
Traceback (most recent call last):
  File "bleReceiver.py", line 135, in closeBluetoothSocket
    self.clientSocket.close()
AttributeError: 'NoneType' object has no attribute 'close'
[2019-10-15 15:29:16,622] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothSocket] : Bluetooth server socket successfully created for RFCOMM service...
[2019-10-15 15:29:16,623] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket bind successfully on host  to PORT_ANY...
[2019-10-15 15:29:16,623] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket put to listening mode successfully ...
[2019-10-15 15:29:16,623] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Waiting for connection on RFCOMM channel 1
[2019-10-15 15:29:16,623] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [advertiseBluetoothService] : Failed to advertise bluetooth services  ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 77, in advertiseBluetoothService
    profiles = [ SERIAL_PORT_PROFILE ],
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 176, in advertise_service
    raise BluetoothError (str (e))
BluetoothError: (2, 'No such file or directory')
[2019-10-15 18:59:48,163] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [acceptBluetoothConnection] : Failed to accept bluetooth connection ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 86, in acceptBluetoothConnection
    self.clientSocket, clientInfo = self.serverSocket.accept()
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 129, in accept
    client, addr = self._sock.accept ()
KeyboardInterrupt
[2019-10-15 18:59:48,163] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [deserializedData] : Failed to de-serialized string ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 113, in deserializedData
    self.dataObj=pickle.loads(_dataRecv)
TypeError: loads() argument 1 must be string, not None
[2019-10-15 18:59:48,164] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : text.json file successfully opened to <open file 'text.json', mode 'w' at 0x7f44dd5d3810>
[2019-10-15 18:59:48,164] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : Content dumped successfully to the text.json file
[2019-10-15 18:59:48,164] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : text.json file successfully closed
[2019-10-15 18:59:48,164] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [closeBluetoothSocket] : Failed to close the bluetooth sockets 
Traceback (most recent call last):
  File "bleReceiver.py", line 135, in closeBluetoothSocket
    self.clientSocket.close()
AttributeError: 'NoneType' object has no attribute 'close'
[2019-10-16 13:06:03,064] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothSocket] : Bluetooth server socket successfully created for RFCOMM service...
[2019-10-16 13:06:03,065] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket bind successfully on host  to PORT_ANY...
[2019-10-16 13:06:03,065] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket put to listening mode successfully ...
[2019-10-16 13:06:03,065] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Waiting for connection on RFCOMM channel 1
[2019-10-16 13:06:03,065] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [advertiseBluetoothService] : Failed to advertise bluetooth services  ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 77, in advertiseBluetoothService
    profiles = [ SERIAL_PORT_PROFILE ],
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 176, in advertise_service
    raise BluetoothError (str (e))
BluetoothError: (2, 'No such file or directory')
[2019-10-16 13:10:59,330] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [acceptBluetoothConnection] : Failed to accept bluetooth connection ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 86, in acceptBluetoothConnection
    self.clientSocket, clientInfo = self.serverSocket.accept()
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 129, in accept
    client, addr = self._sock.accept ()
KeyboardInterrupt
[2019-10-16 13:10:59,332] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [deserializedData] : Failed to de-serialized string ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 113, in deserializedData
    self.dataObj=pickle.loads(_dataRecv)
TypeError: loads() argument 1 must be string, not None
[2019-10-16 13:10:59,335] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : text.json file successfully opened to <open file 'text.json', mode 'w' at 0x7f566f751810>
[2019-10-16 13:10:59,336] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : Content dumped successfully to the text.json file
[2019-10-16 13:10:59,337] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : text.json file successfully closed
[2019-10-16 13:10:59,338] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [closeBluetoothSocket] : Failed to close the bluetooth sockets 
Traceback (most recent call last):
  File "bleReceiver.py", line 135, in closeBluetoothSocket
    self.clientSocket.close()
AttributeError: 'NoneType' object has no attribute 'close'
[2019-10-16 13:14:35,677] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothSocket] : Bluetooth server socket successfully created for RFCOMM service...
[2019-10-16 13:14:35,678] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket bind successfully on host  to PORT_ANY...
[2019-10-16 13:14:35,678] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket put to listening mode successfully ...
[2019-10-16 13:14:35,678] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Waiting for connection on RFCOMM channel 1
[2019-10-16 13:14:35,678] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [advertiseBluetoothService] : Failed to advertise bluetooth services  ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 77, in advertiseBluetoothService
    profiles = [ SERIAL_PORT_PROFILE ],
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 176, in advertise_service
    raise BluetoothError (str (e))
BluetoothError: (2, 'No such file or directory')
[2019-10-16 13:17:51,374] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [acceptBluetoothConnection] : Failed to accept bluetooth connection ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 86, in acceptBluetoothConnection
    self.clientSocket, clientInfo = self.serverSocket.accept()
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 129, in accept
    client, addr = self._sock.accept ()
KeyboardInterrupt
[2019-10-16 13:17:51,377] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [deserializedData] : Failed to de-serialized string ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 113, in deserializedData
    self.dataObj=pickle.loads(_dataRecv)
TypeError: loads() argument 1 must be string, not None
[2019-10-16 13:17:51,382] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : text.json file successfully opened to <open file 'text.json', mode 'w' at 0x7feef68c1810>
[2019-10-16 13:17:51,383] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : Content dumped successfully to the text.json file
[2019-10-16 13:17:51,385] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : text.json file successfully closed
[2019-10-16 13:17:51,386] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [closeBluetoothSocket] : Failed to close the bluetooth sockets 
Traceback (most recent call last):
  File "bleReceiver.py", line 135, in closeBluetoothSocket
    self.clientSocket.close()
AttributeError: 'NoneType' object has no attribute 'close'
[2019-10-16 13:33:03,421] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothSocket] : Bluetooth server socket successfully created for RFCOMM service...
[2019-10-16 13:33:03,422] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket bind successfully on host  to PORT_ANY...
[2019-10-16 13:33:03,423] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket put to listening mode successfully ...
[2019-10-16 13:33:03,423] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Waiting for connection on RFCOMM channel 1
[2019-10-16 13:33:03,423] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [advertiseBluetoothService] : BluetoothServices advertised successfully ...
[2019-10-16 13:33:10,253] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [acceptBluetoothConnection] : Failed to accept bluetooth connection ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 86, in acceptBluetoothConnection
    self.clientSocket, clientInfo = self.serverSocket.accept()
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 129, in accept
    client, addr = self._sock.accept ()
KeyboardInterrupt
[2019-10-16 13:33:10,260] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [deserializedData] : Failed to de-serialized string ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 113, in deserializedData
    self.dataObj=pickle.loads(_dataRecv)
TypeError: loads() argument 1 must be string, not None
[2019-10-16 13:33:10,263] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : text.json file successfully opened to <open file 'text.json', mode 'w' at 0x7fab66ec9810>
[2019-10-16 13:33:10,264] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : Content dumped successfully to the text.json file
[2019-10-16 13:33:10,265] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : text.json file successfully closed
[2019-10-16 13:33:10,266] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [closeBluetoothSocket] : Failed to close the bluetooth sockets 
Traceback (most recent call last):
  File "bleReceiver.py", line 135, in closeBluetoothSocket
    self.clientSocket.close()
AttributeError: 'NoneType' object has no attribute 'close'
[2019-10-16 13:33:56,743] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothSocket] : Bluetooth server socket successfully created for RFCOMM service...
[2019-10-16 13:33:56,744] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket bind successfully on host  to PORT_ANY...
[2019-10-16 13:33:56,744] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket put to listening mode successfully ...
[2019-10-16 13:33:56,744] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Waiting for connection on RFCOMM channel 1
[2019-10-16 13:33:56,744] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [advertiseBluetoothService] : BluetoothServices advertised successfully ...
[2019-10-16 13:34:27,842] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [acceptBluetoothConnection] : Accepted bluetooth connection from ('B8:27:EB:B6:74:28', 1)
[2019-10-16 13:34:27,886] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [recvData] : Data received successfully over bluetooth connection
[2019-10-16 13:34:27,887] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [deserializedData] : Serialized string converted successfully to object
[2019-10-16 13:34:27,888] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : text.json file successfully opened to <open file 'text.json', mode 'w' at 0x7f8d428dd810>
[2019-10-16 13:34:27,889] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : Content dumped successfully to the text.json file
[2019-10-16 13:34:27,890] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : text.json file successfully closed
[2019-10-16 13:34:27,891] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [closeBluetoothSocket] : Bluetooth sockets successfully closed ...
[2019-10-16 13:44:56,134] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothSocket] : Bluetooth server socket successfully created for RFCOMM service...
[2019-10-16 13:44:56,134] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket bind successfully on host  to PORT_ANY...
[2019-10-16 13:44:56,134] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket put to listening mode successfully ...
[2019-10-16 13:44:56,135] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Waiting for connection on RFCOMM channel 1
[2019-10-16 13:44:56,135] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [advertiseBluetoothService] : BluetoothServices advertised successfully ...
[2019-10-16 13:45:02,697] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [acceptBluetoothConnection] : Failed to accept bluetooth connection ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 86, in acceptBluetoothConnection
    self.clientSocket, clientInfo = self.serverSocket.accept()
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 129, in accept
    client, addr = self._sock.accept ()
KeyboardInterrupt
[2019-10-16 13:45:02,698] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [deserializedData] : Failed to de-serialized string ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 113, in deserializedData
    self.dataObj=pickle.loads(_dataRecv)
TypeError: loads() argument 1 must be string, not None
[2019-10-16 13:45:02,699] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : text.json file successfully opened to <open file 'text.json', mode 'w' at 0x7f39d4100810>
[2019-10-16 13:45:02,699] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : Content dumped successfully to the text.json file
[2019-10-16 13:45:02,699] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : text.json file successfully closed
[2019-10-16 13:45:02,699] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [closeBluetoothSocket] : Failed to close the bluetooth sockets 
Traceback (most recent call last):
  File "bleReceiver.py", line 135, in closeBluetoothSocket
    self.clientSocket.close()
AttributeError: 'NoneType' object has no attribute 'close'
[2019-10-16 13:48:19,200] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothSocket] : Bluetooth server socket successfully created for RFCOMM service...
[2019-10-16 13:48:19,200] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket bind successfully on host  to PORT_ANY...
[2019-10-16 13:48:19,201] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket put to listening mode successfully ...
[2019-10-16 13:48:19,201] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Waiting for connection on RFCOMM channel 1
[2019-10-16 13:48:19,201] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [advertiseBluetoothService] : BluetoothServices advertised successfully ...
[2019-10-16 13:49:05,480] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [acceptBluetoothConnection] : Failed to accept bluetooth connection ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 86, in acceptBluetoothConnection
    self.clientSocket, clientInfo = self.serverSocket.accept()
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 129, in accept
    client, addr = self._sock.accept ()
KeyboardInterrupt
[2019-10-16 13:49:05,484] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [deserializedData] : Failed to de-serialized string ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 113, in deserializedData
    self.dataObj=pickle.loads(_dataRecv)
TypeError: loads() argument 1 must be string, not None
[2019-10-16 13:49:05,487] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : text.json file successfully opened to <open file 'text.json', mode 'w' at 0x7fdfd447c810>
[2019-10-16 13:49:05,488] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : Content dumped successfully to the text.json file
[2019-10-16 13:49:05,490] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : text.json file successfully closed
[2019-10-16 13:49:05,491] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [closeBluetoothSocket] : Failed to close the bluetooth sockets 
Traceback (most recent call last):
  File "bleReceiver.py", line 135, in closeBluetoothSocket
    self.clientSocket.close()
AttributeError: 'NoneType' object has no attribute 'close'
[2019-10-16 13:49:57,342] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothSocket] : Bluetooth server socket successfully created for RFCOMM service...
[2019-10-16 13:49:57,343] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket bind successfully on host  to PORT_ANY...
[2019-10-16 13:49:57,343] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket put to listening mode successfully ...
[2019-10-16 13:49:57,343] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Waiting for connection on RFCOMM channel 1
[2019-10-16 13:49:57,344] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [advertiseBluetoothService] : BluetoothServices advertised successfully ...
[2019-10-16 13:50:31,231] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [acceptBluetoothConnection] : Failed to accept bluetooth connection ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 86, in acceptBluetoothConnection
    self.clientSocket, clientInfo = self.serverSocket.accept()
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 129, in accept
    client, addr = self._sock.accept ()
KeyboardInterrupt
[2019-10-16 13:50:31,236] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [deserializedData] : Failed to de-serialized string ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 113, in deserializedData
    self.dataObj=pickle.loads(_dataRecv)
TypeError: loads() argument 1 must be string, not None
[2019-10-16 13:50:31,238] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : text.json file successfully opened to <open file 'text.json', mode 'w' at 0x7fbf2da2a810>
[2019-10-16 13:50:31,239] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : Content dumped successfully to the text.json file
[2019-10-16 13:50:31,240] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : text.json file successfully closed
[2019-10-16 13:50:31,241] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [closeBluetoothSocket] : Failed to close the bluetooth sockets 
Traceback (most recent call last):
  File "bleReceiver.py", line 135, in closeBluetoothSocket
    self.clientSocket.close()
AttributeError: 'NoneType' object has no attribute 'close'
[2019-10-16 13:52:19,092] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothSocket] : Bluetooth server socket successfully created for RFCOMM service...
[2019-10-16 13:52:19,093] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket bind successfully on host  to PORT_ANY...
[2019-10-16 13:52:19,093] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket put to listening mode successfully ...
[2019-10-16 13:52:19,093] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Waiting for connection on RFCOMM channel 1
[2019-10-16 13:52:19,093] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [advertiseBluetoothService] : BluetoothServices advertised successfully ...
[2019-10-16 13:52:45,108] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [acceptBluetoothConnection] : Accepted bluetooth connection from ('B8:27:EB:B6:74:28', 1)
[2019-10-16 13:52:45,166] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [recvData] : Data received successfully over bluetooth connection
[2019-10-16 13:52:45,167] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [deserializedData] : Serialized string converted successfully to object
[2019-10-16 13:52:45,169] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : text.json file successfully opened to <open file 'text.json', mode 'w' at 0x7f5a90005810>
[2019-10-16 13:52:45,171] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : Content dumped successfully to the text.json file
[2019-10-16 13:52:45,172] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : text.json file successfully closed
[2019-10-16 13:52:45,175] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [closeBluetoothSocket] : Bluetooth sockets successfully closed ...
[2019-10-16 14:04:46,938] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothSocket] : Bluetooth server socket successfully created for RFCOMM service...
[2019-10-16 14:04:46,938] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket bind successfully on host  to PORT_ANY...
[2019-10-16 14:04:46,939] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket put to listening mode successfully ...
[2019-10-16 14:04:46,939] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Waiting for connection on RFCOMM channel 1
[2019-10-16 14:04:46,939] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [advertiseBluetoothService] : Failed to advertise bluetooth services  ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 74, in advertiseBluetoothService
    advertise_service( self.serverSocket, self.serviceName,
AttributeError: bleServer instance has no attribute 'serviceName'
[2019-10-16 14:06:08,378] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [acceptBluetoothConnection] : Failed to accept bluetooth connection ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 86, in acceptBluetoothConnection
    self.clientSocket, clientInfo = self.serverSocket.accept()
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 129, in accept
    client, addr = self._sock.accept ()
KeyboardInterrupt
[2019-10-16 14:06:08,383] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [deserializedData] : Failed to de-serialized string ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 113, in deserializedData
    self.dataObj=pickle.loads(_dataRecv)
TypeError: loads() argument 1 must be string, not None
[2019-10-16 14:06:08,385] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : Failed to write json contents to the file ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 121, in writeJsonFile
    jsonFileObj = open(self.jsonFile,"w")
AttributeError: bleServer instance has no attribute 'jsonFile'
[2019-10-16 14:06:08,388] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [closeBluetoothSocket] : Failed to close the bluetooth sockets 
Traceback (most recent call last):
  File "bleReceiver.py", line 135, in closeBluetoothSocket
    self.clientSocket.close()
AttributeError: 'int' object has no attribute 'close'
[2019-10-16 14:06:09,470] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothSocket] : Bluetooth server socket successfully created for RFCOMM service...
[2019-10-16 14:06:09,470] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket bind successfully on host  to PORT_ANY...
[2019-10-16 14:06:09,471] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket put to listening mode successfully ...
[2019-10-16 14:06:09,471] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Waiting for connection on RFCOMM channel 1
[2019-10-16 14:06:09,471] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [advertiseBluetoothService] : Failed to advertise bluetooth services  ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 74, in advertiseBluetoothService
    advertise_service( self.serverSocket, self.serviceName,
AttributeError: bleServer instance has no attribute 'serviceName'
[2019-10-16 14:08:44,578] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [acceptBluetoothConnection] : Failed to accept bluetooth connection ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 86, in acceptBluetoothConnection
    self.clientSocket, clientInfo = self.serverSocket.accept()
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 129, in accept
    client, addr = self._sock.accept ()
KeyboardInterrupt
[2019-10-16 14:08:44,582] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [deserializedData] : Failed to de-serialized string ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 113, in deserializedData
    self.dataObj=pickle.loads(_dataRecv)
TypeError: loads() argument 1 must be string, not None
[2019-10-16 14:08:44,584] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : Failed to write json contents to the file ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 121, in writeJsonFile
    jsonFileObj = open(self.jsonFile,"w")
AttributeError: bleServer instance has no attribute 'jsonFile'
[2019-10-16 14:08:44,586] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [closeBluetoothSocket] : Failed to close the bluetooth sockets 
Traceback (most recent call last):
  File "bleReceiver.py", line 135, in closeBluetoothSocket
    self.clientSocket.close()
AttributeError: 'int' object has no attribute 'close'
[2019-10-16 14:12:35,735] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothSocket] : Bluetooth server socket successfully created for RFCOMM service...
[2019-10-16 14:12:35,735] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket bind successfully on host  to PORT_ANY...
[2019-10-16 14:12:35,735] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket put to listening mode successfully ...
[2019-10-16 14:12:35,736] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Waiting for connection on RFCOMM channel 1
[2019-10-16 14:12:35,736] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [advertiseBluetoothService] : Failed to advertise bluetooth services  ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 74, in advertiseBluetoothService
    advertise_service( self.serverSocket, self.serviceName,
AttributeError: bleServer instance has no attribute 'serviceName'
[2019-10-16 14:19:58,558] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [acceptBluetoothConnection] : Failed to accept bluetooth connection ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 86, in acceptBluetoothConnection
    self.clientSocket, clientInfo = self.serverSocket.accept()
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 129, in accept
    client, addr = self._sock.accept ()
KeyboardInterrupt
[2019-10-16 14:19:58,561] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [deserializedData] : Failed to de-serialized string ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 113, in deserializedData
    self.dataObj=pickle.loads(_dataRecv)
TypeError: loads() argument 1 must be string, not None
[2019-10-16 14:19:58,563] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : Failed to write json contents to the file ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 121, in writeJsonFile
    jsonFileObj = open(self.jsonFile,"w")
AttributeError: bleServer instance has no attribute 'jsonFile'
[2019-10-16 14:19:58,565] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [closeBluetoothSocket] : Failed to close the bluetooth sockets 
Traceback (most recent call last):
  File "bleReceiver.py", line 135, in closeBluetoothSocket
    self.clientSocket.close()
AttributeError: 'int' object has no attribute 'close'
[2019-10-16 14:20:34,703] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothSocket] : Bluetooth server socket successfully created for RFCOMM service...
[2019-10-16 14:20:34,703] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket bind successfully on host  to PORT_ANY...
[2019-10-16 14:20:34,704] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket put to listening mode successfully ...
[2019-10-16 14:20:34,704] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Waiting for connection on RFCOMM channel 1
[2019-10-16 14:20:34,704] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [advertiseBluetoothService] : Failed to advertise bluetooth services  ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 74, in advertiseBluetoothService
    advertise_service( self.serverSocket, self.serviceName,
AttributeError: bleServer instance has no attribute 'serviceName'
[2019-10-16 14:23:28,169] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [acceptBluetoothConnection] : Failed to accept bluetooth connection ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 86, in acceptBluetoothConnection
    self.clientSocket, clientInfo = self.serverSocket.accept()
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 129, in accept
    client, addr = self._sock.accept ()
KeyboardInterrupt
[2019-10-16 14:23:28,171] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [deserializedData] : Failed to de-serialized string ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 113, in deserializedData
    self.dataObj=pickle.loads(_dataRecv)
TypeError: loads() argument 1 must be string, not None
[2019-10-16 14:23:28,173] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : Failed to write json contents to the file ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 121, in writeJsonFile
    jsonFileObj = open(self.jsonFile,"w")
AttributeError: bleServer instance has no attribute 'jsonFile'
[2019-10-16 14:23:28,174] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [closeBluetoothSocket] : Failed to close the bluetooth sockets 
Traceback (most recent call last):
  File "bleReceiver.py", line 135, in closeBluetoothSocket
    self.clientSocket.close()
AttributeError: 'int' object has no attribute 'close'
[2019-10-16 14:23:28,948] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothSocket] : Bluetooth server socket successfully created for RFCOMM service...
[2019-10-16 14:23:28,948] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket bind successfully on host  to PORT_ANY...
[2019-10-16 14:23:28,949] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket put to listening mode successfully ...
[2019-10-16 14:23:28,949] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Waiting for connection on RFCOMM channel 1
[2019-10-16 14:23:28,949] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [advertiseBluetoothService] : Failed to advertise bluetooth services  ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 74, in advertiseBluetoothService
    advertise_service( self.serverSocket, self.serviceName,
AttributeError: bleServer instance has no attribute 'serviceName'
[2019-10-16 14:31:52,693] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [acceptBluetoothConnection] : Failed to accept bluetooth connection ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 86, in acceptBluetoothConnection
    self.clientSocket, clientInfo = self.serverSocket.accept()
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 129, in accept
    client, addr = self._sock.accept ()
KeyboardInterrupt
[2019-10-16 14:31:52,697] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [deserializedData] : Failed to de-serialized string ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 113, in deserializedData
    self.dataObj=pickle.loads(_dataRecv)
TypeError: loads() argument 1 must be string, not None
[2019-10-16 14:31:52,699] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : Failed to write json contents to the file ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 121, in writeJsonFile
    jsonFileObj = open(self.jsonFile,"w")
AttributeError: bleServer instance has no attribute 'jsonFile'
[2019-10-16 14:31:52,701] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [closeBluetoothSocket] : Failed to close the bluetooth sockets 
Traceback (most recent call last):
  File "bleReceiver.py", line 135, in closeBluetoothSocket
    self.clientSocket.close()
AttributeError: 'int' object has no attribute 'close'
[2019-10-16 14:31:53,451] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothSocket] : Bluetooth server socket successfully created for RFCOMM service...
[2019-10-16 14:31:53,451] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket bind successfully on host  to PORT_ANY...
[2019-10-16 14:31:53,451] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket put to listening mode successfully ...
[2019-10-16 14:31:53,451] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Waiting for connection on RFCOMM channel 1
[2019-10-16 14:31:53,452] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [advertiseBluetoothService] : BluetoothServices advertised successfully ...
[2019-10-16 14:31:55,545] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [acceptBluetoothConnection] : Failed to accept bluetooth connection ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 86, in acceptBluetoothConnection
    self.clientSocket, clientInfo = self.serverSocket.accept()
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 129, in accept
    client, addr = self._sock.accept ()
KeyboardInterrupt
[2019-10-16 14:31:55,549] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [deserializedData] : Failed to de-serialized string ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 113, in deserializedData
    self.dataObj=pickle.loads(_dataRecv)
TypeError: loads() argument 1 must be string, not None
[2019-10-16 14:31:55,551] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : text.json file successfully opened to <open file 'text.json', mode 'w' at 0x7fcf7d675810>
[2019-10-16 14:31:55,552] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : Content dumped successfully to the text.json file
[2019-10-16 14:31:55,553] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : text.json file successfully closed
[2019-10-16 14:31:55,554] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [closeBluetoothSocket] : Failed to close the bluetooth sockets 
Traceback (most recent call last):
  File "bleReceiver.py", line 135, in closeBluetoothSocket
    self.clientSocket.close()
AttributeError: 'NoneType' object has no attribute 'close'
[2019-10-16 14:32:44,765] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothSocket] : Bluetooth server socket successfully created for RFCOMM service...
[2019-10-16 14:32:44,765] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket bind successfully on host  to PORT_ANY...
[2019-10-16 14:32:44,765] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket put to listening mode successfully ...
[2019-10-16 14:32:44,765] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Waiting for connection on RFCOMM channel 1
[2019-10-16 14:32:44,766] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [advertiseBluetoothService] : BluetoothServices advertised successfully ...
[2019-10-16 14:32:58,113] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [acceptBluetoothConnection] : Failed to accept bluetooth connection ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 86, in acceptBluetoothConnection
    self.clientSocket, clientInfo = self.serverSocket.accept()
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 129, in accept
    client, addr = self._sock.accept ()
KeyboardInterrupt
[2019-10-16 14:32:58,118] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [deserializedData] : Failed to de-serialized string ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 113, in deserializedData
    self.dataObj=pickle.loads(_dataRecv)
TypeError: loads() argument 1 must be string, not None
[2019-10-16 14:32:58,120] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : text.json file successfully opened to <open file 'text.json', mode 'w' at 0x7f7b5408a810>
[2019-10-16 14:32:58,122] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : Content dumped successfully to the text.json file
[2019-10-16 14:32:58,123] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : text.json file successfully closed
[2019-10-16 14:32:58,123] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [closeBluetoothSocket] : Failed to close the bluetooth sockets 
Traceback (most recent call last):
  File "bleReceiver.py", line 135, in closeBluetoothSocket
    self.clientSocket.close()
AttributeError: 'int' object has no attribute 'close'
[2019-10-16 14:32:58,972] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothSocket] : Bluetooth server socket successfully created for RFCOMM service...
[2019-10-16 14:32:58,973] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket bind successfully on host  to PORT_ANY...
[2019-10-16 14:32:58,973] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket put to listening mode successfully ...
[2019-10-16 14:32:58,973] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Waiting for connection on RFCOMM channel 1
[2019-10-16 14:32:58,974] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [advertiseBluetoothService] : BluetoothServices advertised successfully ...
[2019-10-16 14:35:17,577] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [acceptBluetoothConnection] : Failed to accept bluetooth connection ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 86, in acceptBluetoothConnection
    self.clientSocket, clientInfo = self.serverSocket.accept()
  File "/usr/lib/python2.7/dist-packages/bluetooth/bluez.py", line 129, in accept
    client, addr = self._sock.accept ()
KeyboardInterrupt
[2019-10-16 14:35:17,578] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [deserializedData] : Failed to de-serialized string ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 113, in deserializedData
    self.dataObj=pickle.loads(_dataRecv)
TypeError: loads() argument 1 must be string, not None
[2019-10-16 14:35:17,579] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : text.json file successfully opened to <open file 'text.json', mode 'w' at 0x7f999fa25810>
[2019-10-16 14:35:17,579] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : Content dumped successfully to the text.json file
[2019-10-16 14:35:17,579] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : text.json file successfully closed
[2019-10-16 14:35:17,579] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [closeBluetoothSocket] : Failed to close the bluetooth sockets 
Traceback (most recent call last):
  File "bleReceiver.py", line 135, in closeBluetoothSocket
    self.clientSocket.close()
AttributeError: 'int' object has no attribute 'close'
[2019-10-16 14:35:19,238] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothSocket] : Bluetooth server socket successfully created for RFCOMM service...
[2019-10-16 14:35:19,238] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket bind successfully on host  to PORT_ANY...
[2019-10-16 14:35:19,238] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket put to listening mode successfully ...
[2019-10-16 14:35:19,238] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Waiting for connection on RFCOMM channel 1
[2019-10-16 14:35:19,239] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [advertiseBluetoothService] : BluetoothServices advertised successfully ...
[2019-10-16 14:36:34,252] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [acceptBluetoothConnection] : Accepted bluetooth connection from ('B8:27:EB:B6:74:28', 1)
[2019-10-16 14:36:34,295] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [recvData] : Data received successfully over bluetooth connection
[2019-10-16 14:36:34,297] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [deserializedData] : Serialized string converted successfully to object
[2019-10-16 14:36:34,298] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : text.json file successfully opened to <open file 'text.json', mode 'w' at 0x7fc155e5a810>
[2019-10-16 14:36:34,299] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : Content dumped successfully to the text.json file
[2019-10-16 14:36:34,300] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : text.json file successfully closed
[2019-10-16 14:36:34,301] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [closeBluetoothSocket] : Bluetooth sockets successfully closed ...
[2019-10-16 14:37:26,574] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothSocket] : Bluetooth server socket successfully created for RFCOMM service...
[2019-10-16 14:37:26,574] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket bind successfully on host  to PORT_ANY...
[2019-10-16 14:37:26,574] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket put to listening mode successfully ...
[2019-10-16 14:37:26,575] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Waiting for connection on RFCOMM channel 1
[2019-10-16 14:37:26,575] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [advertiseBluetoothService] : BluetoothServices advertised successfully ...
[2019-10-16 14:37:45,576] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [acceptBluetoothConnection] : Accepted bluetooth connection from ('B8:27:EB:B6:74:28', 1)
[2019-10-16 14:37:45,657] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [recvData] : Data received successfully over bluetooth connection
[2019-10-16 14:37:45,658] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [deserializedData] : Serialized string converted successfully to object
[2019-10-16 14:37:45,659] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : text.json file successfully opened to <open file 'text.json', mode 'w' at 0x7fc2813b9810>
[2019-10-16 14:37:45,661] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : Content dumped successfully to the text.json file
[2019-10-16 14:37:45,662] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : text.json file successfully closed
[2019-10-16 14:37:45,664] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [closeBluetoothSocket] : Bluetooth sockets successfully closed ...
[2019-10-16 14:40:17,228] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothSocket] : Bluetooth server socket successfully created for RFCOMM service...
[2019-10-16 14:40:17,228] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket bind successfully on host  to PORT_ANY...
[2019-10-16 14:40:17,229] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket put to listening mode successfully ...
[2019-10-16 14:40:17,229] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Waiting for connection on RFCOMM channel 1
[2019-10-16 14:40:17,229] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [advertiseBluetoothService] : BluetoothServices advertised successfully ...
[2019-10-16 14:40:47,686] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [acceptBluetoothConnection] : Accepted bluetooth connection from ('B8:27:EB:B6:74:28', 1)
[2019-10-16 14:40:47,744] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [recvData] : Data received successfully over bluetooth connection
[2019-10-16 14:40:47,745] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [deserializedData] : Serialized string converted successfully to object
[2019-10-16 14:40:47,747] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : text.json file successfully opened to <open file 'text.json', mode 'w' at 0x7fda89d9c810>
[2019-10-16 14:40:47,748] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : Content dumped successfully to the text.json file
[2019-10-16 14:40:47,749] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : text.json file successfully closed
[2019-10-16 14:40:47,750] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [closeBluetoothSocket] : Bluetooth sockets successfully closed ...
[2019-10-16 14:42:19,872] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothSocket] : Bluetooth server socket successfully created for RFCOMM service...
[2019-10-16 14:42:19,872] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket bind successfully on host  to PORT_ANY...
[2019-10-16 14:42:19,872] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket put to listening mode successfully ...
[2019-10-16 14:42:19,873] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Waiting for connection on RFCOMM channel 1
[2019-10-16 14:42:19,873] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [advertiseBluetoothService] : BluetoothServices advertised successfully ...
[2019-10-16 14:42:37,360] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [acceptBluetoothConnection] : Accepted bluetooth connection from ('B8:27:EB:B6:74:28', 1)
[2019-10-16 14:42:37,423] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [recvData] : Data received successfully over bluetooth connection
[2019-10-16 14:42:37,425] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [deserializedData] : Serialized string converted successfully to object
[2019-10-16 14:42:37,425] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : text.json file successfully opened to <open file 'text.json', mode 'w' at 0x7f4106bc5810>
[2019-10-16 14:42:37,426] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : Content dumped successfully to the text.json file
[2019-10-16 14:42:37,426] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : text.json file successfully closed
[2019-10-16 14:42:37,427] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [closeBluetoothSocket] : Bluetooth sockets successfully closed ...
[2019-10-16 15:17:37,028] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothSocket] : Bluetooth server socket successfully created for RFCOMM service...
[2019-10-16 15:17:37,028] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket bind successfully on host  to PORT_ANY...
[2019-10-16 15:17:37,028] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket put to listening mode successfully ...
[2019-10-16 15:17:37,028] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Waiting for connection on RFCOMM channel 1
[2019-10-16 15:17:37,029] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [advertiseBluetoothService] : BluetoothServices advertised successfully ...
[2019-10-16 15:18:02,223] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [acceptBluetoothConnection] : Accepted bluetooth connection from ('B8:27:EB:B6:74:28', 1)
[2019-10-16 15:18:02,258] : [ERROR] : [bleServerLogger] : [bleReceiver.py] : [deserializedData] : Failed to de-serialized string ... 
Traceback (most recent call last):
  File "bleReceiver.py", line 117, in deserializedData
    self.dataObj=pickle.loads(_dataRecv)
TypeError: loads() argument 1 must be string, not None
[2019-10-16 15:18:02,260] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : text.json file successfully opened to <open file 'text.json', mode 'w' at 0x7efc99318810>
[2019-10-16 15:18:02,261] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : Content dumped successfully to the text.json file
[2019-10-16 15:18:02,262] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : text.json file successfully closed
[2019-10-16 15:18:02,263] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [closeBluetoothSocket] : Bluetooth sockets successfully closed ...
[2019-10-16 15:26:22,201] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothSocket] : Bluetooth server socket successfully created for RFCOMM service...
[2019-10-16 15:26:22,201] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket bind successfully on host  to PORT_ANY...
[2019-10-16 15:26:22,201] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket put to listening mode successfully ...
[2019-10-16 15:26:22,201] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Waiting for connection on RFCOMM channel 1
[2019-10-16 15:26:22,202] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [advertiseBluetoothService] : BluetoothServices advertised successfully ...
[2019-10-16 15:26:51,177] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [acceptBluetoothConnection] : Accepted bluetooth connection from ('B8:27:EB:B6:74:28', 1)
[2019-10-16 15:26:51,313] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [recvData] : Data received successfully over bluetooth connection
[2019-10-16 15:26:51,314] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [deserializedData] : Serialized string converted successfully to object
[2019-10-16 15:26:51,315] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : text.json file successfully opened to <open file 'text.json', mode 'w' at 0x7f654c6fd810>
[2019-10-16 15:26:51,317] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : Content dumped successfully to the text.json file
[2019-10-16 15:26:51,318] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : text.json file successfully closed
[2019-10-16 15:26:51,319] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [closeBluetoothSocket] : Bluetooth sockets successfully closed ...
[2019-10-29 15:30:51,564] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothSocket] : Bluetooth server socket successfully created for RFCOMM service...
[2019-10-29 15:30:51,565] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket bind successfully on host  to PORT_ANY...
[2019-10-29 15:30:51,566] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket put to listening mode successfully ...
[2019-10-29 15:30:51,567] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Waiting for connection on RFCOMM channel 1
[2019-10-29 15:30:51,568] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [advertiseBluetoothService] : BluetoothServices advertised successfully ...
[2019-10-29 15:32:05,235] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [acceptBluetoothConnection] : Accepted bluetooth connection from ('B8:27:EB:B6:74:28', 1)
[2019-10-29 15:32:05,358] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [recvData] : Data received successfully over bluetooth connection
[2019-10-29 15:32:05,359] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [deserializedData] : Serialized string converted successfully to object
[2019-10-29 15:32:05,361] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : text.json file successfully opened to <open file 'text.json', mode 'w' at 0x7f726c4b9810>
[2019-10-29 15:32:05,362] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : Content dumped successfully to the text.json file
[2019-10-29 15:32:05,363] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : text.json file successfully closed
[2019-10-29 15:32:05,364] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [closeBluetoothSocket] : Bluetooth sockets successfully closed ...
[2019-10-29 18:32:47,983] : [INFO] : [bleClientLogger] : [bleSender.py] : [<module>] : Setup logging configuration
[2019-10-29 18:32:47,984] : [INFO] : [bleClientLogger] : [bleSender.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-29 18:33:01,154] : [INFO] : [bleClientLogger] : [bleSender.py] : [getBluetoothServices] : Re-connecting  Bluetooth services : 2 attempt
[2019-10-29 18:33:12,766] : [INFO] : [bleClientLogger] : [bleSender.py] : [getBluetoothServices] : Re-connecting  Bluetooth services : 3 attempt
[2019-10-29 18:33:12,767] : [ERROR] : [bleClientLogger] : [bleSender.py] : [getBluetoothServices] : Couldn't find the RaspberryPi Bluetooth service : Invalid uuid
Traceback (most recent call last):
  File "bleSender.py", line 59, in getBluetoothServices
    if not bleService: raise SystemExit(), None
SystemExit
[2019-10-29 18:33:12,770] : [INFO] : [bleClientLogger] : [bleSender.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-29 18:43:14,272] : [INFO] : [bleClientLogger] : [bleSender.py] : [<module>] : Setup logging configuration
[2019-10-29 18:43:14,272] : [INFO] : [bleClientLogger] : [bleSender.py] : [getBluetoothServices] : Searching for  Bluetooth services ...
[2019-10-29 18:43:25,790] : [INFO] : [bleClientLogger] : [bleSender.py] : [getBluetoothServices] : Re-connecting  Bluetooth services : 2 attempt
[2019-10-29 18:43:38,756] : [INFO] : [bleClientLogger] : [bleSender.py] : [getBluetoothServices] : Re-connecting  Bluetooth services : 3 attempt
[2019-10-29 18:43:38,769] : [INFO] : [bleClientLogger] : [bleSender.py] : [getBluetoothSocket] : Bluetooth client socket successfully created for RFCOMM service  ...
[2019-10-30 11:00:48,689] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothSocket] : Bluetooth server socket successfully created for RFCOMM service...
[2019-10-30 11:00:48,690] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket bind successfully on host  to PORT_ANY...
[2019-10-30 11:00:48,690] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Bluetooth server socket put to listening mode successfully ...
[2019-10-30 11:00:48,690] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [getBluetoothConnection] : Waiting for connection on RFCOMM channel 1
[2019-10-30 11:00:48,690] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [advertiseBluetoothService] : BluetoothServices advertised successfully ...
[2019-10-30 11:01:10,748] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [acceptBluetoothConnection] : Accepted bluetooth connection from ('B8:27:EB:B6:74:28', 1)
[2019-10-30 11:01:10,785] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [recvData] : Data received successfully over bluetooth connection
[2019-10-30 11:01:10,786] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [deserializedData] : Serialized string converted successfully to object
[2019-10-30 11:01:10,787] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : text.json file successfully opened to <open file 'text.json', mode 'w' at 0x7f3de1944810>
[2019-10-30 11:01:10,788] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : Content dumped successfully to the text.json file
[2019-10-30 11:01:10,789] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [writeJsonFile] : text.json file successfully closed
[2019-10-30 11:01:10,791] : [INFO] : [bleServerLogger] : [bleReceiver.py] : [closeBluetoothSocket] : Bluetooth sockets successfully closed ...
